
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>Chapter 9: Phosphate transfer reactions &#8212; Organic Chemistry with a Biological Emphasis</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="../_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" href="../_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Chapter 10: Nucleophilic addition to carbonyl groups" href="Chapter%2010%20-%20Nucleophilic%20Addition%20to%20Carbonyl%20Groups.html" />
    <link rel="prev" title="&lt;no title&gt;" href="../Introduction.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../index.html">
      
      
      
      <h1 class="site-logo" id="site-title">Organic Chemistry with a Biological Emphasis</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../Introduction.html">
                    <no title>
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Organic Chemistry with a Biological Emphasis Part 2
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   Chapter 9: Phosphate transfer reactions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%2010%20-%20Nucleophilic%20Addition%20to%20Carbonyl%20Groups.html">
   Chapter 10: Nucleophilic addition to carbonyl groups
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Solutions
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../Solutions/Solutions%20to%20in-chapter%20problems.html">
   Solutions to In-Chapter Exercises
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../Solutions/Solutions%20to%20end-of-chapter%20problems.html">
   Solutions to selected end-of-chapter problems
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-repository-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Source repositories">
      <i class="fab fa-github"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="https://github.com/c-bun/OCBE-Textbook"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Source repository"
>
  

<span class="headerbtn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="headerbtn__text-container">repository</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/c-bun/OCBE-Textbook/issues/new?title=Issue%20on%20page%20%2FOCBE_II/Chapter 9 - Phosphate Transfer Reactions.html&body=Your%20issue%20content%20here."
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Open an issue"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="headerbtn__text-container">open issue</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/c-bun/OCBE-Textbook/edit/main/book/OCBE_II/Chapter 9 - Phosphate Transfer Reactions.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Edit this page"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-pencil-alt"></i>
  </span>
<span class="headerbtn__text-container">suggest edit</span>
</a>

      </li>
      
    </ul>
  </div>
</div>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="../_sources/OCBE_II/Chapter 9 - Phosphate Transfer Reactions.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#introduction">
   Introduction
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-1-overview-of-phosphate-groups">
   Section 9.1: Overview of phosphate groups
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-terms-and-abbreviations">
     9.1A: Terms and abbreviations
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-acid-constants-and-protonation-states">
     9.1B: Acid constants and protonation states
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-bonding-in-phosphates">
     9.1C: Bonding in phosphates
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-2-phosphate-transfer-reactions-an-overview">
   Section 9.2: Phosphate transfer reactions - an overview
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-3-atp-the-principal-phosphate-donor">
   Section 9.3: ATP, the principal phosphate donor
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-4-phosporylation-of-alcohols">
   Section 9.4: Phosporylation of alcohols
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-5-phosphorylation-of-carboxylates">
   Section 9.5: Phosphorylation of carboxylates
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-6-hydrolysis-of-organic-phosphates">
   Section 9.6: Hydrolysis of organic phosphates
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-7-phosphate-diesters-in-dna-and-rna">
   Section 9.7: Phosphate diesters in DNA and RNA
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-8-the-organic-chemistry-of-genetic-engineering">
   Section 9.8: The organic chemistry of genetic engineering
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-9-nmr-of-phosphorylated-compounds">
   Section 9.9: NMR of phosphorylated compounds
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#key-concepts-to-review">
   Key concepts to review
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#problems">
   Problems
  </a>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Chapter 9: Phosphate transfer reactions</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#introduction">
   Introduction
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-1-overview-of-phosphate-groups">
   Section 9.1: Overview of phosphate groups
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-terms-and-abbreviations">
     9.1A: Terms and abbreviations
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-acid-constants-and-protonation-states">
     9.1B: Acid constants and protonation states
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-bonding-in-phosphates">
     9.1C: Bonding in phosphates
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-2-phosphate-transfer-reactions-an-overview">
   Section 9.2: Phosphate transfer reactions - an overview
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-3-atp-the-principal-phosphate-donor">
   Section 9.3: ATP, the principal phosphate donor
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-4-phosporylation-of-alcohols">
   Section 9.4: Phosporylation of alcohols
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-5-phosphorylation-of-carboxylates">
   Section 9.5: Phosphorylation of carboxylates
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-6-hydrolysis-of-organic-phosphates">
   Section 9.6: Hydrolysis of organic phosphates
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-7-phosphate-diesters-in-dna-and-rna">
   Section 9.7: Phosphate diesters in DNA and RNA
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-8-the-organic-chemistry-of-genetic-engineering">
   Section 9.8: The organic chemistry of genetic engineering
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-9-9-nmr-of-phosphorylated-compounds">
   Section 9.9: NMR of phosphorylated compounds
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#key-concepts-to-review">
   Key concepts to review
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#problems">
   Problems
  </a>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="chapter-9-phosphate-transfer-reactions">
<h1>Chapter 9: Phosphate transfer reactions<a class="headerlink" href="#chapter-9-phosphate-transfer-reactions" title="Permalink to this headline">#</a></h1>
<img alt="../_images/image2.jpeg" src="../_images/image2.jpeg" />
<p><strong>Mono Lake, California</strong></p>
<p>(photo credit https://www.flickr.com/photos/slolane/)</p>
<section id="introduction">
<h2>Introduction<a class="headerlink" href="#introduction" title="Permalink to this headline">#</a></h2>
<p>This chapter is about the chemistry of phosphates, a ubiquitous
functional group in biomolecules that is based on phosphoric acid:</p>
<img alt="../_images/image3.png" src="../_images/image3.png" />
<p>fig 1d</p>
<p>In late 2010, people around the world found themselves getting a crash
course in phosphate chemistry as they watched the evening news. Those
who paid close attention to the developing story also got an interesting
glimpse into the world of scientific research and debate.</p>
<p>It all started when the American National Aeronautics and Space
Administration (NASA) released the following statement to the news
media/OCBE_II:</p>
<blockquote>
<div><p>“NASA will hold a news conference at 2 p.m. EST on Thursday, Dec. 2,
to</p>
</div></blockquote>
<p>discuss an astrobiology finding that will impact the search for evidence
of</p>
<p>extraterrestrial life.”</p>
<p>The wording of the statement attracted widespread media/OCBE_II attention, and
had some people holding their breath in anticipation that NASA would be
introducing a newly discovered alien life form to the world. When
December 2<sup>nd</sup> came, however, those hoping to meet ET were
disappointed – the life form being introduced was a bacterium, and it
was from our own planet. To biologists and chemists, though, the
announcement was nothing less than astounding.</p>
<p>The NASA scientists worked hard to emphasize the significance of their
discovery during the news conference. Dr. Felicia Wolfe-Simon, a young
postdoctoral researcher who had spearheaded the project, stated that
they had “cracked open the door to what’s possible for life elsewhere in
the universe - and that’s profound”. A senior NASA scientist claimed
that their results would “fundamentally change how we define life”, and,
in attempting to convey the importance of the discovery to a reporter
from the newspaper USA Today, referred to an episode from the original
Star Trek television series in which the crew of the Starship Enterprise
encounters a race of beings whose biochemistry is based on silica rather
than carbon.</p>
<p>The new strain of bacteria, dubbed ‘GFAJ-1’, had been isolated from the
arsenic-rich mud surrounding salty, alkaline Mono Lake in central
California. What made the strain so unique, according to the NASA team,
was that it had evolved the ability to substitute arsenate for phosphate
in its DNA. Students of biology and chemistry know that phosphorus is
one of the six elements that are absolutely required for life as we know
it, and that DNA is a polymer linked by phosphate groups. Arsenic, which
is directly below phosphorus on the periodic table, is able to assume a
bonding arrangement like that of phosphate, so it might seem reasonable
to wonder whether arsenate could replace phosphate in DNA and other
biological molecules. Actually finding a living thing with
arsenate-linked DNA would indeed be a momentous achievement in biology,
as this would represent a whole new chemistry for the most fundamental
molecule of life, and would change our understanding of the chemical
requirements for life to exist on earth - and potentially other planets.</p>
<p>In 1987, Professor F.H. Westheimer of Harvard University published what
would become a widely read commentary in Science Magazine entitled “Why
Nature Chose Phosphates”. In it, he discussed the chemical properties
that make the phosphate group so ideal for the many roles that it plays
in biochemistry, chief among them the role of a linker group for DNA
polymers. One of the critical characteristics of phosphate that
Westheimer pointed out was that the bonds linking phosphate to organic
molecules are stable in water. Clearly, if you are selecting a
functional group to link your DNA, you don’t want to choose one that
will rapidly break apart in water. Among the functional groups that
Westheimer compared to phosphate in terms of its suitability as a
potential DNA linker was arsenate –but he very quickly dismissed the
idea of arsenate-linked DNA because it would be far too unstable in
water.</p>
<p>Given this background, it is not hard to imagine that many scientists
were puzzled, to say the least, by the NASA results. While the popular
media/OCBE_II took the announcement at face value and excitedly reported the
results as a monumental discovery – NASA is, after all, a highly
respected scientific body and the study was being published in Science
Magazine, one of the most prestigious scientific journals in the world –
many scientists quickly voiced their skepticism, mainly in the
relatively new and unconstrained venue of the blogosphere.
Microbiologist Rosie Redfield of the University of British Columbia,
writing in her blog devoted to ‘open science’, wrote a detailed and
highly critical analysis of the study. She pointed out, among other
things, that the experimenters had failed to perform the critical
purification and mass spectrometry analyses needed to demonstrate that
arsenate was indeed being incorporated into the DNA backbone, and that
the broth in which the bacteria were being grown actually contained
enough phosphate for them to live and replicate using normal
phosphate-linked DNA. Science journalist Carl Zimmer, in a column in the
online magazine Slate, contacted twelve experts to get their opinions,
and they were overwhelmingly negative. One of the experts said bluntly,
“This paper should not have been published”. Basically, the NASA
researchers were making an astounding claim that, if true, would refute
decades of established knowledge about the chemistry of DNA – but the
evidence they presented was far from convincing. Carl Sagan’s widely
quoted dictum - “extraordinary claims require extraordinary evidence” -
seemed to apply remarkably well to the situation.</p>
<p>What followed was a very public, very lively, and not always completely
collegial debate among scientists about the proper way to discuss
science: the NASA researchers appeared to dismiss the criticism amassed
against their study because it came from blogs, websites, and Twitter
feeds. The proper venue for such discussion, they claimed, was in the
peer-reviewed literature. Critics countered that their refusal to
respond to anything outside of the traditional peer-review system was
disingenuous, because they had made full use of the publicity-generating
power of the internet and mainstream media/OCBE_II in the first place when they
announced their results with such fanfare.</p>
<p>The traditional venue for debate, while quite a bit slower than the
blogosphere, did eventually come through. When the full paper was
published in Science a few months later, it was accompanied by eight
‘technical comments’ from other researchers pointing out deficiencies in
the study, an ‘editors note’, and a broader news article about the
controversy. In July of 2012, a paper was published in Science under the
title “GFAJ-1 Is an Arsenate-Resistant, Phosphate-Dependent Organism”.
The paper reported definitive evidence that DNA from GFAJ-1, under the
conditions described in the NASA paper, did <em>not</em> have arsenate
incorporated into its structure. Just like professor Westheimer
discussed in the 1980s, it appears that nature really did choose
phosphate – and only phosphate – after all … at least on this
planet.</p>
<p><u>Background reading and viewing:</u></p>
<p>Youtube video of the NASA press conference:</p>
<blockquote>
<div><p>http://www.youtube.com/watch?v=WVuhBt03z8g.</p>
</div></blockquote>
<p>Wolfe-Simon, F. et al. <em>Science Express</em>, Dec 2, 2010. The first preview
article on the proposed ‘arsenic bacteria’.</p>
<p>Wolfe-Simon, F. et al., <em>Science</em> <strong>2011</strong>, <em>332</em>, 1163. The full
research paper in Science Magazine.</p>
<p>Westheimer, F.H. <em>Science</em> <strong>1987</strong>, <em>235</em>, 1173. The article by
Westheimer titled ‘Why Nature Chose Phosphates’.</p>
<p>Zimmer, Carl, Slate, Dec 7, 2010: Blog post by Carl Zimmer titled ‘This
Paper Should Not Have Been Published’.</p>
<p>http://www.slate.com/articles/health_and_science/science/2010/12/this_paper_should_not_have_been_published.html</p>
<p>Redfield, R. Blog post Dec 4, 2010:</p>
<p>http://rrresearch.fieldofscience.com/2010/12/arsenic-associated-bacteria-nasas.html</p>
<p><em>Science</em> <strong>2012</strong>, <em>337</em>, 467. The paper in Science Magazine refuting
the validity of the arsenic bacteria claim.</p>
</section>
<section id="section-9-1-overview-of-phosphate-groups">
<h2>Section 9.1: Overview of phosphate groups<a class="headerlink" href="#section-9-1-overview-of-phosphate-groups" title="Permalink to this headline">#</a></h2>
<p>Phosphate is everywhere in biochemistry. As we were reminded in the
introduction to this chapter, our DNA is linked by phosphate:</p>
<img alt="../_images/image4.png" src="../_images/image4.png" />
<p>fig 1a</p>
<p>The function of many proteins is regulated - switched on and off - by
enzymes which attach or remove a phosphate group from the side chains of
serine, threonine, or tyrosine residues.</p>
<img alt="../_images/image5.png" src="../_images/image5.png" />
<p>fig 1b</p>
<p>Countless diseases are caused by defects in phosphate transferring
enzymes. As just one example, achondroplasia, a common cause of
dwarfism, is caused by a defect in an enzyme whose function is to
transfer a phosphate to a tyrosine residue in a growth-related signaling
protein.</p>
<p>Finally, phosphates are excellent leaving groups in biological organic
reactions, as we will see many times throughout the remainder of this
book.</p>
<p>Clearly, an understanding of phosphate chemistry is central to the study
of biological organic chemistry. We’ll begin with an overview of terms
used when talking about phosphates.</p>
<section id="a-terms-and-abbreviations">
<h3>9.1A: Terms and abbreviations<a class="headerlink" href="#a-terms-and-abbreviations" title="Permalink to this headline">#</a></h3>
<p>The fully deprotonated conjugate base of phosphoric acid is called a
phosphate ion, or <strong>inorganic phosphate</strong> (often abbreviated
‘P*<sub>i</sub><em>’). When two phosphate groups are linked to each other,
the linkage itself is referred to as a <strong>‘phosphate anhydride’</strong>, and
the compound is called <strong>‘inorganic pyrophosphate’</strong> (often abbreviated
PP</em><sub>i</sub>*).</p>
<img alt="../_images/image6.png" src="../_images/image6.png" />
<p>fig 1</p>
<p>The chemical linkage between phosphate and a carbon atom is a
<strong>phosphate ester</strong>. Adenosine monophosphate (AMP) has a single
phosphate ester linkage.</p>
<img alt="../_images/image7.png" src="../_images/image7.png" />
<p>fig 2</p>
<p>Adenosine triphosphate has one phosphate ester linkage and two phosphate
anhydride linkages.</p>
<img alt="../_images/image8.png" src="../_images/image8.png" />
<p>fig 3</p>
<p>Oxygen atoms in phosphate groups are referred to either as
<strong>‘bridging’</strong> or <strong>‘non-bridging’</strong>, depending on their position. An
organic diphosphate has two bridging oxygens (one in the phosphate ester
linkage and one in the phosphate anhydride linkage) and five
non-bridging oxygens:</p>
<img alt="../_images/image9.png" src="../_images/image9.png" />
<p>fig 4</p>
<p>A single phosphate is linked to <em>two</em> organic groups is called
<strong>phosphate diester</strong>. The backbone of DNA is linked by phosphate
diesters.</p>
<img alt="../_images/image10.png" src="../_images/image10.png" />
<p>fig 5</p>
<p>Organic phosphates are often abbreviated using ‘OP’ and ‘OPP’ for mono-
and diphosphates, respectively. For example, glucose-6-phosphate and
isopentenyl diphosphate are often depicted as shown below. Notice that
the ‘<em>P</em>’ abbreviation includes the associated oxygen atoms and negative
charges.</p>
<img alt="../_images/image11.png" src="../_images/image11.png" />
<p>fig 6</p>
<p><u>Exercise 9.1</u>: Consider the biological compounds below, some of
which are shown with abbreviated structures:</p>
<img alt="../_images/image12.png" src="../_images/image12.png" />
<p>fig 4a</p>
<p>a) Which contain one or more phosphate anhydride linkages? Specify the
number of phosphate anhydride linkages in your answers.</p>
<p>b) Which contain one or more phosphate monoesters? Again, specify the
number for each answer.</p>
<p>c) Which contain a phosphate diester?</p>
<p>d) Which could be described as an organic diphosphate?</p>
<p>e) For each compound, specify the number of bridging and non-bridging
oxygens in the phosphate group.</p>
<u>  
</u>
</section>
<section id="b-acid-constants-and-protonation-states">
<h3>9.1B: Acid constants and protonation states<a class="headerlink" href="#b-acid-constants-and-protonation-states" title="Permalink to this headline">#</a></h3>
<p>Phosphoric acid is triprotic, meaning that it has three acidic protons
available to donate, with pK<sub>a</sub> values of 1.0, 6.5, and 13.0,
respectively. (da Silva and Williams)</p>
<img alt="../_images/image13.png" src="../_images/image13.png" />
<p>fig 7</p>
<p>These acid constant values, along with the Henderson-Hasselbalch
equation (section 7.2C) tell us that, at the physiological pH of
approximately 7, somewhat more than half of the phosphate species will
be in the HPO<sub>4</sub><sup>-2</sup> state, and slightly less than
half will be in the H<sub>2</sub>PO<sub>4</sub><sup>-1</sup> state,
meaning that the average net charge is between -1.5 and -2.0.</p>
<p>Phosphate diesters have a pKa of about 1, meaning that they carry a full
negative charge at physiological pH.</p>
<img alt="../_images/image14.png" src="../_images/image14.png" />
<p>fig 7a</p>
<p>Organic monophosphates, diphosphates, and triphosphates all have net
negative charges and are partially protonated at physiological pH, but
by convention are usually drawn in the fully deprotonated state.</p>
<p><u>Exercise 9.2</u>: Explain why the second pKa of phosphoric acid is so
much higher than the first pKa.</p>
<p><u>Exercise 9.3</u>: What is the approximate net charge of inorganic
phosphate in a solution buffered to pH 1?</p>
<p>Recall from section 8.4 that good leaving groups in organic reactions
are, as a rule, weak bases. In laboratory organic reactions, leaving
groups are often halides or toluenesulfonates (section 8.4), both of
which are weak bases. <strong>In biological organic reactions, phosphates are
very common leaving groups</strong>. These could be inorganic phosphate,
inorganic pyrophosphate, or organic monophosphates, all of which are
weakly basic, especially when coordinated to metal cations such as
Mg<sup>+2</sup> in the active site of an enzyme. We will see many
examples of phosphate leave groups in this and subsequent chapters.</p>
</section>
<section id="c-bonding-in-phosphates">
<h3>9.1C: Bonding in phosphates<a class="headerlink" href="#c-bonding-in-phosphates" title="Permalink to this headline">#</a></h3>
<p>Looking at the location of phosphorus on the periodic table, you might
expect it to bond and react in a fashion similar to nitrogen, which is
located just above it in the same column. Indeed, <strong>phosphines</strong> -
phosphorus analogs of amines - are commonly used in the organic
laboratory.</p>
<img alt="../_images/image15.png" src="../_images/image15.png" />
<p>fig 8</p>
<p>However it is in the form of <em>phosphate</em>, rather than phosphine, that
phosphorus plays its main role in biology.</p>
<p>The four oxygen substituents in phosphate groups are arranged about the
central phosphorus atom with <em>tetrahedral geometry</em>, however there are a
total of <em>five</em> bonds to phosphorus - four σ bonds and one delocalized π
bond.</p>
<img alt="../_images/image16.png" src="../_images/image16.png" />
<p>fig 9</p>
<p>Phosphorus can break the ‘octet rule’ because it is on the third row of
the periodic table, and thus has <em>d</em> orbitals available for bonding. The
minus 3 charge on a fully deprotonated phosphate ion is spread evenly
over the four oxygen atoms, and each phosphorus-oxygen bond can be
considered to have 25% double bond character: in other words, the bond
order is 1.25.</p>
<p>Recall from section 2.1 the hybrid bonding picture for the tetrahedral
nitrogen in an amine group: a single 2<em>s</em> and three 2<em>p</em> orbitals
combine to form four <em>sp<sup>3</sup></em> hybrid orbitals, three of which
form σ bonds and one of which holds a lone pair of electrons.</p>
<img alt="../_images/image17.png" src="../_images/image17.png" />
<p>fig 10</p>
<p>In the hybrid orbital picture for phosphate ion, a single 3<em>s</em> and three
3<em>p</em> orbitals also combine to form four <em>sp<sup>3</sup></em> hybrid orbitals
with tetrahedral geometry. In contrast to an amine, however, four of the
five valance electrons on phosphorus occupy <em>sp<sup>3</sup></em> orbitals,
and the fifth occupies an unhybridized 3<em>d</em> orbital.</p>
<img alt="../_images/image18.png" src="../_images/image18.png" />
<p>fig 11</p>
<p>This orbital arrangement allows for four σ bonds with tetrahedral
geometry in addition to a fifth, delocalized π bond formed by π overlap
between the half-filled 3<em>d</em> orbital on phosphorus and 2<em>p</em> orbitals on
the oxygen atoms.</p>
<p>In phosphate esters, diesters, and anhydrides the π bonding is
delocalized primarily over the <em>non-bridging bonds</em>, while the bridging
bonds have mainly single-bond character. In a phosphate diester, for
example, the two non-bridging oxygens share a -1 charge, as illustrated
by the two major resonance contributors below. The bonding order for the
bridging P-O bonds in a phosphate diester group is about 1, and for the
non-bridging P-O bonds about 1.5. In the resonance contributors in which
the bridging oxygens are shown as double bonds (to the right in the
figure below), there is an additional separation of charge - thus these
contributors are minor and make a relatively unimportant contribution to
the overall bonding picture.</p>
<img alt="../_images/image19.png" src="../_images/image19.png" />
<p>fig 12</p>
<p><u>Exercise 9.4:</u> Draw all of the resonance structures showing the
delocalization of charge on a (fully deprotonated) organic
monophosphate. If a ‘bond order’ of 1.0 is a single bond, and a bond
order of 2.0 is a double bond, what is the approximate bond order of
bridging and non-bridging P-O bonds?</p>
<p>Throughout this book, phosphate groups will often be drawn without
attempting to show tetrahedral geometry, and π bonds and negative
charges will usually be shown localized to a single oxygen. This is done
for the sake of simplification - however it is important always to
remember that the phosphate group is really tetrahedral, the negative
charges are delocalized over the non-bridging oxygens, and that there is
some degree of protonation at physiological pH (with the exception of
the phosphate diester group).</p>
</section>
</section>
<section id="section-9-2-phosphate-transfer-reactions-an-overview">
<h2>Section 9.2: Phosphate transfer reactions - an overview<a class="headerlink" href="#section-9-2-phosphate-transfer-reactions-an-overview" title="Permalink to this headline">#</a></h2>
<p>In a phosphate transfer reaction, a phosphate group is transferred from
a <strong>phosphate group donor</strong> molecule to a <strong>phosphate group acceptor</strong>
molecule:</p>
<img alt="../_images/image20.png" src="../_images/image20.png" />
<p>fig 13</p>
<p>A very important aspect of biological phosphate transfer reactions is
that the electrophilicity of the phosphorus atom is usually enhanced by
the Lewis acid (electron-accepting) effect of one or more magnesium
ions. Phosphate transfer enzymes generally contain a Mg<sup>2+</sup> ion
bound in the active site in a position where it can interact with
non-bridging phosphate oxygens on the substrate. The magnesium ion pulls
electron density <em>away</em> from the phosphorus atom, making it more
electrophilic.</p>
<img alt="../_images/image21.png" src="../_images/image21.png" />
<p>fig 17</p>
<p>Without this metal ion interaction, a phosphate is actually a poor
electrophile, as the negatively-charged oxygens shield the phosphorus
center from attack by a nucleophile.</p>
<p>Note: For the sake of simplicity and clarity, we will not draw the
magnesium ion or other active site groups interacting with phosphate
oxygens in most figures in this chapter - but it is important to keep in
mind that these interactions play an integral role in phosphate transfer
reactions.</p>
<p>Mechanistically speaking, a phosphate transfer reaction at a phosphorus
center can be though of as much like a S<sub>N</sub>2 reaction at a
carbon center. Just like in an S<sub>N</sub>2 reaction, the nucleophile
in a phosphoryl transfer approaches the electrophilic center from the
backside, opposite the leaving group:</p>
<p>**<br />
**</p>
<p><strong>Concerted model:</strong></p>
<img alt="../_images/image22.png" src="../_images/image22.png" />
<p>fig 17a</p>
<p>As the nucleophile gets closer and the leaving group begins its
departure, the bonding geometry at the phosphorus atom changes from
tetrahedral to trigonal bipyramidal at the pentavalent (5-bond)
transition state. As the phosphorus-nucleophile bond gets shorter and
the phosphorus-leaving group bond grows longer, the bonding picture
around the phosphorus atom returns to its original tetrahedral state,
but the stereochemical configuration has been ‘flipped’, or inverted.</p>
<p>In the trigonal bipyramidal transition state, the five substituents are
not equivalent: the three non-bridging oxygens are said to be
<strong>equatorial</strong> (forming the base of a trigonal bipyramid), while the
nucleophile and the leaving group are said to be <strong>apical</strong> (occupying
the tips of the two pyramids).</p>
<img alt="../_images/image23.png" src="../_images/image23.png" />
<p>fig 17b</p>
<p>Although stereochemical inversion in phosphoryl transfer reactions is
predicted by theory, the fact that phosphoryl groups are achiral made it
impossible to observe the phenomenon directly until 1978, when a group
of researchers was able to synthesize organic phosphate esters in which
stable oxygen isotopes <sup>17</sup>O and <sup>18</sup>O were
specifically incorporated. This created a chiral phosphate center.</p>
<img alt="../_images/image24.png" src="../_images/image24.png" />
<p>fig 17c</p>
<p>Subsequent experiments with phosphoryl transfer-catalyzing enzymes
confirmed that these reactions proceed with stereochemical inversion.
(<em>Nature</em> <strong>1978</strong> <em>275</em>, 564; <em>Ann Rev Biochem</em> <strong>1980</strong> <em>49</em>, 877).</p>
<p>The concerted (S<sub>N</sub>2-like) is not the only mechanism that has
been proposed for these reactions - in fact, two other possible
mechanisms have been suggested. In an alternative two-step mechanistic
model, the nucleophile could attack first, forming</p>
<p>a pentavalent, trigonal bipyramidal <em>intermedia/OCBE_IIte</em> (as apposed to a
pentavalent <em>transition state</em>). The reaction is completed when the
leaving group is expelled. The intermedia/OCBE_IIte species would occupy an
energy <em>valley</em> between the two transition states.</p>
<p>**<br />
**</p>
<p><strong>Addition-elimination model:</strong></p>
<img alt="../_images/image25.png" src="../_images/image25.png" />
<p>fig 17e</p>
<p>This is often referred to as an <strong>‘addition-elimination</strong>’ mechanism -
the nucleophile <em>adds</em> to the phosphate first, forming a pentavalent
intermedia/OCBE_IIte, and then the leaving group is <em>eliminated</em>.</p>
<p>An addition-elimination mechanism with a pentavalent intermedia/OCBE_IIte is
<em>not</em> possible for an S<sub>N</sub>2 reaction at a carbon center,
because carbon, as a second-row element, does not have any <em>d</em> orbitals
and cannot form five bonds. Phosphorus, on the other hand, is a
third-row element and is quite capable of forming more than four bonds.
Phosphorus pentachloride, after all, is a stable compound that has five
bonds to chlorine arranged in trigonal bipyramidal geometry around the
central phosphorus.</p>
<img alt="../_images/image26.png" src="../_images/image26.png" />
<p>fig 17f</p>
<p>The phosphorus atom in PCl<sub>5</sub> (and in the hypothetical
pentavalent intermedia/OCBE_IIte pictured above) is considered to be
<strong>sp<sup>3</sup>d</strong> <strong>hybridized</strong>:</p>
<img alt="../_images/image27.png" src="../_images/image27.png" />
<p>fig 17g</p>
<p>There is a third possibility: the reaction could proceed in an
S<sub>N</sub>1-like manner: in other words, <strong>elimination-addition</strong>. In
this model, the phosphorus-leaving group bond breaks first, resulting in
a <strong>metaphosphate</strong> intermedia/OCBE_IIte. This intermedia/OCBE_IIte, which corresponds
to the carbocation intermedia/OCBE_IIte in an S<sub>N</sub>1 reaction and like a
carbocation has trigonal planar geometry, is then attacked by the
nucleophile to form the reaction product.</p>
<p><strong>Elimination-addition model:</strong></p>
<img alt="../_images/image28.png" src="../_images/image28.png" />
<p>fig 5</p>
<p>fig 17h</p>
<p>So which mechanistic model - concerted (S<sub>N</sub>2-like),
addition-elimination, or elimination-addition - best describes enzymatic
phosphate transfer reactions? Chemists love to investigate and debate
questions like this! Just like with the S<sub>N</sub>1/S<sub>N</sub>2
argument discussed in chapter 8, it really boils down to one question:
which happens first, bond-forming or bond-breaking - or do these two
events occur at the same time? From the evidence accumulated to date, it
appears that many enzymatic phosphate transfer reactions can best be
described by the concerted model, although there is still argument about
this, and still many unanswered questions about other details of how
these reactions are catalyzed in active sites. Considering the
importance of phosphate transfer reactions in metabolic pathways, this
area is clearly a very promising one for further investigation. If you
are interesting in learning more about this research, a great place to
start is a review article written by Professor Daniel Herschlag at
Stanford University (<em>Annu. Rev. Biochem</em>. <strong>2011</strong>, <em>80</em>, 669).</p>
<p>For the sake of simplicity and clarity, phosphoryl transfers in this
text will be depicted using the concerted model.</p>
<u>  
Exercise 9.5</u>: Predict the approximate angles between the two bonds
indicated in a phosphate transfer transition state. O<sub>a</sub> refers
to an oxygen at the apical position, and O<sub>e</sub> to an oxygen in
the equatorial position.
<blockquote>
<div><p>a) O<sub>a</sub>-P-O<sub>a</sub> b) O<sub>a</sub>-P-O<sub>e</sub>
c)O<sub>e</sub>-P-O<sub>e</sub></p>
</div></blockquote>
</section>
<section id="section-9-3-atp-the-principal-phosphate-donor">
<h2>Section 9.3: ATP, the principal phosphate donor<a class="headerlink" href="#section-9-3-atp-the-principal-phosphate-donor" title="Permalink to this headline">#</a></h2>
<p>Thus far we have been very general in our discussion of phosphate
transfer reactions, referring only to generic ‘donor’ and ‘acceptor’
species. It’s time to get more specific. The most important donor of
phosphate groups in the cell is a molecule called <strong>adenosine
triphosphate</strong>, commonly known by its abbreviation <strong>ATP</strong>.</p>
<img alt="../_images/image29.png" src="../_images/image29.png" />
<p>fig 21</p>
<p>Notice that there are essentially three parts to the ATP molecule: an
adenine nucleoside ‘base’, a five-carbon sugar (ribose), and
triphosphate. The three phosphates are designated by Greek letters α, β,
and γ, with the α phosphate being the one closest to the ribose.
Adenosine diphosphate (ADP) and adenosine monophosphate (AMP) are also
important players in the reactions of this chapter.</p>
<p>ATP is a big molecule, but the bond-breaking and bond-forming events we
will be studying in this chapter all happen in the phosphate part of the
molecule. You will see structural drawings of ATP, ADP, and AMP
abbreviated in many different ways in this text and throughout the
biochemical literature, depending on what is being illustrated. For
example, the three structures below are all abbreviated depictions of
ATP:</p>
<img alt="../_images/image30.png" src="../_images/image30.png" />
<p>fig 22</p>
<p>The following exercise will give you some practice in recognizing
different abbreviations for ATP and other biological molecules that
contain phosphate groups.</p>
<p><u>Exercise 9.6 :</u> Below are a number of representations, labeled
A-S, of molecules that contain phosphate groups. Different abbreviations
are used. Arrange A-S into groups of drawings that depict the same
species (for example, group together all of the abbreviations which
depict ATP).</p>
<img alt="../_images/image31.png" src="../_images/image31.png" />
<img alt="../_images/image32.png" src="../_images/image32.png" />
<img alt="../_images/image33.png" src="../_images/image33.png" />
<img alt="../_images/image34.png" src="../_images/image34.png" />
<img alt="../_images/image35.png" src="../_images/image35.png" />
<img alt="../_images/image36.png" src="../_images/image36.png" />
<p>You are probably familiar with the physiological role of ATP from your
biology classes - it is commonly called ‘the energy currency of the
cell’. What this means is that ATP stores energy we get from the
oxidation of fuel molecules such as carbohydrates or fats. The energy in
ATP is stored in the two high-energy phosphate anhydride linkages.</p>
<img alt="../_images/image37.png" src="../_images/image37.png" />
<p>fig 23</p>
<p>When one or both of these phosphate anhydride links are broken as a
phosphate group is transferred to an acceptor, a substantial amount of
energy is released. The negative charges on the phosphate groups are
separated, eliminating some of electrostatic repulsion that existed in
ATP. One way to picture this is as a coil springing open, releasing
potential energy.</p>
<img alt="../_images/image38.png" src="../_images/image38.png" />
<p>fig 24</p>
<p>In addition, cleavage of a phosphate anhydride bond means that
surrounding water molecules are able to form more stabilizing
hydrogen-bonding interactions with the products than was possible with
the starting materials, again making the reaction more ‘downhill’, or
exergonic.</p>
<p>It is important to understand that while the phosphate anhydride bonds
in ATP are <em>thermodynamically</em> unstable (they contain a great deal of
chemical energy), they are at the same time <em>kinetically</em> stable:
ATP-cleaving reactions are exothermic, but also have a high energy
barrier, making them very slow <em>unless catalyzed by an enzyme</em>. In other
words, the release of the energy contained in ATP is highly energetic
but also subject to tight control by the interaction of highly evolved
enzymes in our metabolic pathways.</p>
<p>ATP is a versatile phosphate group donor: depending on the site of
nucleophilic attack (at the α, β, or γ phosphorus), different phosphate
transfer outcomes are possible. Below are the three most common patterns
seen in the central metabolic pathways. A ‘squigly’ line in each figure
indicates the P-O bond being broken. We will study specific examples of
each of these in the coming sections.</p>
<p>Attack at the γ-phosphate:</p>
<img alt="../_images/image39.png" src="../_images/image39.png" />
<p>Attack at the β-phosphate:</p>
<img alt="../_images/image40.png" src="../_images/image40.png" />
<p>Attack at the α-phosphate:</p>
<img alt="../_images/image41.png" src="../_images/image41.png" />
<p>fig 24c</p>
<p>The common thread running through all of the ATP-dependent reactions we
will see in this section is the idea that the phosphate acceptor
molecule is undergoing a thermodynamically ‘uphill’ transformation to
become a more reactive species. The energy for this uphill
transformation comes from breaking a high-energy phosphate anhydride
bond in ATP. That is why ATP is often referred to as ‘energy currency’:
the energy in its anhydride bonds is used to ‘pay for’ a
thermodynamically uphill chemical step.</p>
<p><u>Exercise 9.7</u>: Propose a fourth hypothetical phosphate transfer
reaction between ATP and the generic acceptor molecule in the figure
above, in which inorganic phosphate (P<em>i)</em> is a by-product.</p>
<p><u>Exercise 9.8</u> : Why is this hypothetical phosphate transfer
reaction less energetically favorable compared to all of the possible
ATP-cleaving reactions shown in the figure above?</p>
<img alt="../_images/image42.png" src="../_images/image42.png" />
<p>fig 24d</p>
<p>**<br />
**</p>
</section>
<section id="section-9-4-phosporylation-of-alcohols">
<h2>Section 9.4: Phosporylation of alcohols<a class="headerlink" href="#section-9-4-phosporylation-of-alcohols" title="Permalink to this headline">#</a></h2>
<p>A broad family of enzymes called <strong>kinases</strong> catalyze transfer of a
phosphate group from ATP to an alcohol acceptor. Mechanistically, the
alcohol oxygen acts as a nucleophile, attacking the electrophilic
γ-phosphorus of ATP and expelling ADP.</p>
<p>Glucose is phosphorylated in the first step of the glycolysis pathway by
the enzyme hexose kinase (EC 2.7.1.1), forming glucose-6-phosphate.</p>
<p>Hexose kinase mechanism</p>
<img alt="../_images/image43.png" src="../_images/image43.png" />
<p><a class="reference external" href="https://www.youtube.com/watch?v=K9C5OOwJN_M">video tutorial</a></p>
<p>fig 26</p>
<p>Here is a shorthand way to depict this reaction. Notice the “ATP in, ADP
out” notation used below, indicating that ATP is one of the reactants
and ADP is one of the products. From here on, we will frequently use
this common convention to indicate reaction participants whose
structures are not drawn out in a figure.</p>
<img alt="../_images/image44.png" src="../_images/image44.png" />
<p>fig 26b</p>
<p>The biological activity of many proteins is regulated by <strong>protein
kinases</strong>. In a protein kinase reaction, the side chain hydroxyl groups
on serine, threonine, or tyrosine residues of certain proteins are
phosphorylated by ATP:</p>
<img alt="../_images/image45.png" src="../_images/image45.png" />
<p>fig 27</p>
<p>The conversion of a neutral hydroxyl group to a charged phosphate
represents a very dramatic change in the local architecture of the
protein, potentially altering its folding pattern and ability to bind to
small molecules or other proteins. A protein’s biological function can
be ‘switched on’ by phosphorylation of a single residue, and switched
off again by removal of the phosphate group. The latter reaction we will
examine later in this chapter.</p>
<p><u>Exercise 9.9</u>:</p>
<p>a) Draw a curved-arrow mechanism, using abbreviations as appropriate,
for the serine kinase reaction.</p>
<p>b) Threonine kinase catalyzes the phosphorylation of the side chain
hydroxyl group of threonine residues in proteins. Draw the structure,
including the configuration of all stereocenters, of a phosphothreonine
residue. Explain how you can predict the stereochemistry of the side
chain.</p>
<p>Although stereochemical inversion in phosphate transfer is predicted by
theory, the fact that phosphate groups are achiral made it impossible
for a long time to verify the phenomenon directly. This was finally
accomplished in the late 1970’s, when a group of researchers
demonstrated phosphate inversion in kinase enzymes using chemically
synthesized ATP in which three different isotopes of oxygen were
incorporated into the γ−phosphate, thus creating a chiral phosphorus
center. <em>(Ann. Rev. Biochem</em>. <strong>1980</strong> <em>49</em>, 877).</p>
<img alt="../_images/image46.png" src="../_images/image46.png" />
<p>fig 27a</p>
<p>Alcohols can be converted into organic <strong>di</strong>phosphates in two different
ways. A two-step process simply involves successive transfers of the
γ-phosphate groups of two ATP donors, such as in these sequential steps
in isoprenoid biosynthesis. (EC 2.7.1.36; EC 2.7.4.2). A compound called
mevalonate is diphosphorylated in this way in the early phase of the
biosynthetic pathway for cholesterol, steroid hormones, and other
isoprenoid molecules.</p>
<img alt="../_images/image47.png" src="../_images/image47.png" />
<p>fig 28</p>
<p>The mechanism for the first phosphorylation step is analogous to that
for an alcohol kinase reaction, which we have just seen. In the second
phosphate transfer step, catalyzed by a separate enzyme, one of the
phosphate oxygens on the organic monophosphate acts as a nucleophilic
phosphate acceptor, attacking the γ-phosphate of a second ATP.</p>
<img alt="../_images/image48.png" src="../_images/image48.png" />
<p>fig 29a</p>
<p>In some metabolic pathways, diphosphorylation occurs by a different
mechanism from the one above. Instead of sequentially transferring two
phosphates from two ATP donors, the alternate mechanism occurs in a
single step: the nucleophilic acceptor molecule attacks the β-phosphate
of ATP, rather than the γ-phosphate. After formation of the trigonal
bipyramidal intermedia/OCBE_IIte, it is AMP (not ADP) which is expelled, and
what started out as the β and γ phosphates of ATP both remain with the
acceptor.</p>
<p>In the biosynthesis of DNA and RNA nucleotides, one of the hydroxyl
groups on ribose-5-phosphate is diphosphorylated (EC 2.7.6.1) in a
one-step mechanism:</p>
<p>A one-step alcohol diphosphorylation reaction (PRPP synthase):</p>
<img alt="../_images/image49.png" src="../_images/image49.png" />
<p>Mechanism:</p>
<img alt="../_images/image50.png" src="../_images/image50.png" />
<p>fig 31, 31b</p>
<p>The metabolic role of both of the diphosphorylation processes we just
saw is to convert a hydroxyl group into a good leaving group (recall
that hydroxide ions are strong bases and poor leaving groups, while
phosphates/diphosphates, especially when stabilized in an enzyme active
site, are weak bases and very good leaving groups). In nucleoside
biosynthesis pathways, the diphosphate group of PRPP acts as a leaving
group in the very next metabolic step, which is an S<sub>N</sub>1
reaction: we have already seen this reaction in section 8.7B).</p>
</section>
<section id="section-9-5-phosphorylation-of-carboxylates">
<h2>Section 9.5: Phosphorylation of carboxylates<a class="headerlink" href="#section-9-5-phosphorylation-of-carboxylates" title="Permalink to this headline">#</a></h2>
<p>Thus far we have seen hydroxyl oxygens and phosphate oxygens acting as
nucleophilic accepting groups in ATP-dependent phosphate transfer
reactions. Carboxylate oxygens can also accept phosphate groups from
ATP. This typically happens in two different ways. First, the
carboxylate can attack the γ-phosphate of ATP to accept phosphate,</p>
<p>generates a species known as an ‘acyl phosphate’. An example is the
first part of the reaction catalyzed by glutamine synthase (EC 6.3.1.2):</p>
<img alt="../_images/image51.png" src="../_images/image51.png" />
<p>Alternatively, carboxylate groups are often converted into a species
referred to as an ‘acyl-AMP’ . Here, the carboxylate oxygen attacks the
α-phosphate of ATP leading to release of inorganic pyrophosphate. An
example is the first part of the reaction catalyzed by the enzyme
asparagine synthetase: (EC 6.3.5.4):</p>
<img alt="../_images/image52.png" src="../_images/image52.png" />
<p>fig 35</p>
<p><u>Exercise 9.10</u>: Draw a curved-arrow mechanism for the phosphate
transfer reaction shown below (EC 2.7.2.3), which is from the glycolysis
pathway. Note that ADP is on the reactant side and ATP is a product (the
opposite of what we have seen so far). Hint: What functional group is
the nucleophile? What functional group is the leaving group?</p>
<img alt="../_images/image53.png" src="../_images/image53.png" />
<p>fig 36</p>
</section>
<section id="section-9-6-hydrolysis-of-organic-phosphates">
<h2>Section 9.6: Hydrolysis of organic phosphates<a class="headerlink" href="#section-9-6-hydrolysis-of-organic-phosphates" title="Permalink to this headline">#</a></h2>
<p>While kinase enzymes catalyze the phosphorylation of organic compounds,
enzymes called <strong>phosphatases</strong> catalyze <em>de</em>phosphorylation reactions.
<strong>The reactions catalyzed by kinases and phosphatases are <em>not</em> the
reverse of one another:</strong> kinases irreversibly transfer phosphate groups
from ATP (or sometimes other nucleoside triphosphates) to various
organic acceptor compounds, while <em>phosphatases transfer phosphate
groups from organic compounds to water</em>: these are <em>hydrolysis</em>
reactions. Kinase reactions involve an inherently ‘uphill’ step
(phosphorylation of an alcohol, for example) being paid for with an
inherently ‘downhill’ step (cleavage of an anhydride bond in ATP).
Phosphatase reactions, on the other hand, are thermodynamically
‘downhill’, and while they require an enzyme to speed them up, they do
not involve ‘spending’ energy currency the way kinase reactions do.</p>
<p>Phosphatase reaction:</p>
<img alt="../_images/image54.png" src="../_images/image54.png" />
<p>fig 37</p>
<p>There are two possible general mechanisms for a phosphatase reaction.
Some enzymes catalyze <em>direct</em> hydrolysis reactions, in which the
phosphate group is removed by direct attack of a water molecule at the
phosphate center:</p>
<p>Phosphatase mechanism (direct hydrolysis):</p>
<img alt="../_images/image55.png" src="../_images/image55.png" />
<p>fig 37</p>
<p>One of the two phosphate groups on fructose 1,6-bisphosphate is
hydrolyzed in such a way late in the gluconeogenesis pathway.
(<em>Biochemistry</em> <strong>2000</strong>, <em>39</em>, 8565; EC 3.1.3.11)</p>
<img alt="../_images/image56.png" src="../_images/image56.png" />
<p>fig 37b</p>
<p>Many phosphatase reactions, however, operate by a slightly more
complicated mechanism than what is shown above. In the first phase, a
nucleophilic enzyme group (typically a cysteine, aspartate, glutamate,
or histidine side chain, designated in the figure below as ‘X’) attacks
the phosphate group. In the second phase, the phosphorylated residue is
hydrolized. For example, protein tyrosine phosphatase catalyzes the
dephosphorylation of phosphotyrosine residues in some proteins - this is
the other half of the regulatory ‘on-off switch’ that we discussed
earlier in the context of protein kinases. In the first step, the
phosphate group is directly donated to a cysteine side chain in the
phosphatase enzyme’s active site. In the second step, the
phosphocysteine intermedia/OCBE_IIte is cleaved by water to form inorganic
phosphate and regenerate the free cysteine in the active site.</p>
<p>Indirect phosphatase reaction:</p>
<p>Step 1:</p>
<img alt="../_images/image57.png" src="../_images/image57.png" />
<p>Step 2:</p>
<img alt="../_images/image58.png" src="../_images/image58.png" />
<p>fig 38d</p>
<p>Notice that in the end, the phosphate group has still been transferred
to a water molecule, albeit indirectly. How would you know, just by
looking at the substrate and product of the protein tyrosine phosphatase
reaction, that the phosphate is not transferred <em>directly</em> to a water
molecule? Simply put, you <em>wouldn’t</em> know this information without the
benefit of knowledge gained from biochemical experimentation.</p>
<p><u>Exercise 9.11</u>: If you were to look just at the substrates and
products of a phosphatase reaction without knowing anything about the
mechanism, it is apparent that a <em>nucleophilic substitution</em> mechanism
could theoretically account for the products formed. Draw out a
hypothetical nucleophilic substitution mechanism for the hydrolysis of a
phosphoserine residue and show how researchers, by running the reaction
in H<sub>2</sub><sup>18</sup>O, (isotopically labeled water), could
potentially distinguish between a nucleophilic substitution and
phosphate group transfer mechanism by looking at where the
<sup>18</sup>O atom ends up in the products.</p>
</section>
<section id="section-9-7-phosphate-diesters-in-dna-and-rna">
<h2>Section 9.7: Phosphate diesters in DNA and RNA<a class="headerlink" href="#section-9-7-phosphate-diesters-in-dna-and-rna" title="Permalink to this headline">#</a></h2>
<p>Phosphate diesters play an absolutely critical role in nature - they are
the molecular ‘tape’ that connect the individual nucleotides in DNA and
RNA <em>via</em> a sugar-phosphate backbone. Take note of the 1’ - 5’ carbon
numbering shown below for the ribose sugar - these numbers will be used
frequently in the coming discussion. The ‘prime’ symbol (’) is used to
distinguish the ribose carbon numbers from the nucleotide base carbon
numbers (which are not shown here).</p>
<img alt="../_images/image59.png" src="../_images/image59.png" />
<p>fig 5</p>
<p>The introduction to this chapter referenced a widely-read 1987
commentary in Science Magazine, in which F.H. Westheimer of Harvard
University addressed the question of why phosphates were ‘chosen’ by
nature for critical biochemical job of linking DNA (<em>Science</em> <strong>1987</strong>,
<em>235</em>, 1173). He emphasizes how critical it is for the phosphate diester
linkage in DNA to be stable in water – in other words, it must be
resistant to spontaneous (nonenzymatic) hydrolysis. Even very infrequent
occurrence of such an undesired hydrolysis event could be disastrous for
an organism, given that an intact DNA strand is a <em>long-term</em> storage
mechanism for genetic information.</p>
<p>Westheimer pointed out that the inherent stability of DNA is a due in
large part to the negative charge on the non-bridging oxygen of the
phosphate diester linker, which effectively repels nucleophilic water
molecules and shields the electrophilic phosphorus atom from attack.</p>
<img alt="../_images/image60.png" src="../_images/image60.png" />
<p>fig 40</p>
<p>While DNA is quite stable with regard to spontaneous hydrolysis, it of
course can be degraded by specific <em>enzymatic</em> hydrolysis, where the
phosphate electrophile is activated for attack through noncovalent
interactions (eg. with Mg<sup>2+)</sup> in the active site. Enzymes that
hydrolyze the phosphate diester bonds in DNA are called nucleases, and
we will learn more about them in section 9.8.</p>
<p>Unlike DNA, RNA is quite vulnerable to spontaneous hydrolysis in aqueous
solution. This does not present a physiological dilemma, because the
function of RNA is to encode genetic information on a temporary rather
than long-term basis. Why does hydrolysis occur so much more rapidly in
RNA than in DNA? The answer has everything to do with the lowered
<em>entropic</em> barrier to the reaction (you might want to quickly review the
concept of entropy at this point). RNA nucleotides, unlike the
<em><strong>deoxy</strong></em>nucleotides of DNA, have a hydroxyl group at the neighboring
2’ carbon. The 2’ hydroxyl group is right next to the electrophilic
phosphorus atom, poised in a good position to make a nucleophilic
attack, breaking the RNA chain and forming a cyclic phosphate diester
intermedia/OCBE_IIte (see figure below).</p>
<img alt="../_images/image61.png" src="../_images/image61.png" />
<p>fig 41</p>
<p>Researchers working with RNA have to be careful to store their samples
at very cold temperatures, preferably freeze-dried or precipitated in
ethanol, to avoid hydrolysis. The problem of RNA decomposition is
compounded by the fact that RNAase enzymes, which catalyze RNA
hydrolysis, are present on the surface of human skin and are very
stable, long-lived, and difficult to destroy.</p>
<p>In contrast, DNA samples can be safely stored in aqueous buffer in a
refrigerator, or in a freezer for longer-term storage.</p>
</section>
<section id="section-9-8-the-organic-chemistry-of-genetic-engineering">
<h2>Section 9.8: The organic chemistry of genetic engineering<a class="headerlink" href="#section-9-8-the-organic-chemistry-of-genetic-engineering" title="Permalink to this headline">#</a></h2>
<p>Many enzymes that catalyze reactions involving the phosphate diester
bonds of DNA have been harnessed for use in genetic engineering -
techniques in which we copy, snip, and splice DNA in order to create
custom versions of genes. The tools of genetic engineering have become
indispensable and commonplace in the past decade, and most researchers
working on the biological side of chemistry use them extensively. The
days of painstakingly purifying an enzyme from bacterial cultures or
ground-up cow livers are pretty much gone. Now scientists clone the gene
that encodes the enzyme, make any desired changes (by site-directed
mutagenesis, for example), and use a host such as <em>E. coli</em> or yeast to
produce ‘recombinant’ enzyme from the cloned gene. You will learn the
details of many of these procedures in a biochemistry or molecular
biology course. What we will focus on now is applying what we have
learned about phosphate group transfer reactions so that we can
recognize some of the organic chemistry that is happening in a cloning
experiment.</p>
<p>The first thing you have to do in a gene cloning procedure is to copy a
DNA strand. This is accomplished by an enzyme called <strong>DNA polymerase</strong>
(EC 2.7.7.7), which uses a single strand of DNA as a template to
synthesize a second, complementary strand (the full picture of this
complex process is well beyond the scope of this book, but recall that
we talked about the discovery of thermostable DNA polymerase in the
introduction to chapter 6).</p>
<p>You may have learned in a biology class that DNA is synthesized in the
3’ to 5’ direction. Notice below that the 3’ hydroxyl group on the end
of the growing DNA strand attacks the α-phosphate of a
2’-deoxynucleoside triphosphate (dNTP), expelling inorganic
pyrophosphate.</p>
<p>DNA polymerase reaction:</p>
<img alt="../_images/image62.png" src="../_images/image62.png" />
<p>fig 42</p>
<p>Scientists are able to cut DNA using ‘molecular scissor’ enzymes called
<strong>restriction endonucleases</strong> that cleave double-stranded DNA by
hydrolysis at specific base sequences.</p>
<p>DNA hydrolysis by restriction endonucleases:</p>
<img alt="../_images/image63.png" src="../_images/image63.png" />
<p>fig 43</p>
<p>Notice that the result of this hydrolytic cleavage reaction is one
segment of DNA with a hydroxy group at the 3’ position, and a second
segment with a phosphate group at the 5’ position.</p>
<p>A commonly used restriction endonuclease called ‘BamHI’ cleaves
double-stranded DNA specifically at the following 6-base sequence:</p>
<img alt="../_images/image64.png" src="../_images/image64.png" />
<p>fig 43</p>
<p>Notice that a ‘staggered’ cut is made: this is a common (and useful)
property of many endonucleases, although some make ‘blunt-ended’ cuts.</p>
<p>While an endonuclease cleaves a phosphodiester linkage in a DNA strand,
<strong>DNA ligase</strong> (EC 6.5.1.1) accomplishes the reverse process: it
catalyzes the formation of a new 3’-5’ link between two strands:</p>
<p>DNA ligase reaction:</p>
<img alt="../_images/image65.png" src="../_images/image65.png" />
<p>fig 45a</p>
<p>Note that there is initially no leaving group on the 5’ phosphate of
DNA<sub>2</sub>, which makes a direct phosphate transfer reaction
impossible. The strategy employed by the DNA ligase enzyme is to first
<em>activate</em> the 5’ phosphate of DNA<sub>2</sub> using ATP (phase 1
below), then the ligation reaction can proceed (phase 2)</p>
<p>DNA ligation</p>
<p>Phase 1: activation</p>
<img alt="../_images/image66.png" src="../_images/image66.png" />
<p>Phase 2: ligation:</p>
<img alt="../_images/image67.png" src="../_images/image67.png" />
<p>One more enzymatic tool in the genetic engineering arsenal bears
mention. In some cloning procedures, a researcher may want to prevent
unwanted ligation of DNA. This can be accomplished by using the enzyme
<strong>alkaline phosphatase</strong> (EC 3.1.3.1), which catalyzes the
dephosphorylation of many different organic phosphates, including
5’-phosphorylated DNA (recall that we discussed phosphatases in section
9.6).</p>
<p>Alkaline phosphatase reaction:</p>
<img alt="../_images/image68.png" src="../_images/image68.png" />
<p>fig 46</p>
<p>With the phosphate group removed, ligation is impossible - there is no
way to make a new phosphodiester bond without a 5’ phosphate group!</p>
<p>**<br />
**</p>
</section>
<section id="section-9-9-nmr-of-phosphorylated-compounds">
<h2>Section 9.9: NMR of phosphorylated compounds<a class="headerlink" href="#section-9-9-nmr-of-phosphorylated-compounds" title="Permalink to this headline">#</a></h2>
<p>Because so many biological molecules contain phosphoryl groups, it is
worthwhile to look at how scientists use NMR to determine the structure
of these molecules. Recall from section 5.1 that <sup>31</sup>P, the
most abundant isotope of phosphorus, is NMR active: it can be directly
observed by <sup>31</sup>P-NMR, and indirectly observed in
<sup>1</sup>H-NMR and <sup>13</sup>C-NMR through its spin-coupling
interactions with neighboring protons and carbons, respectively.</p>
<p>Consider the case of isopentenyl diphosphate, the building block
molecule used by cells to make ‘isoprenoid’ compounds such as
cholesterol (in many animals), or β-carotene (in some plants). NMR
spectra of this molecule were taken in a D<sub>2</sub>O solvent,
buffered with ND<sub>4</sub>OD (the deuterium equivalent of aqueous
ammonium hydroxide, NH<sub>4</sub>OH) (<em>J. Org. Chem</em>. <strong>1986</strong>, <em>51</em>,
4768). In our discussion, carbon atoms are specified with numbers,
protons with lower case letters, and phosphorus atoms with upper case
letters.</p>
<img alt="../_images/image69.png" src="../_images/image69.png" />
<p>First, let’s look at the proton spectrum:</p>
<p><u><sup>1</sup>H-NMR</u></p>
<p>H<sub>a</sub>: 4.05 ppm (td); <sup>3</sup>J<sub>Ha-Hb</sub> = 6.6 Hz;
<sup>3</sup>J<sub>Ha-PA</sub> = 3.3 Hz.</p>
<p>H<sub>b</sub>: 2.39 ppm (t) <sup>3</sup>J<sub>Ha-Hb</sub> = 6.6 Hz</p>
<p>H<sub>c</sub>: 4.86 ppm (s)</p>
<p>H<sub>d</sub>: 1.77 ppm (s)</p>
<img alt="../_images/image70.png" src="../_images/image70.png" />
<p>The signals for H<sub>b</sub>, H<sub>c</sub>, and H<sub>d</sub> look
like we would expect from our discussion in chapter 5, with the
exception of H<sub>c</sub> which you will be invited to discuss in the
exercise below.</p>
<p>Why, though, is the signal for H<sub>a</sub> split into a <em>triplet of
doublets</em> (td)? First of all, as, expected, the two neighboring
H<sub>b</sub> protons split the H<sub>a</sub> signal into a triplet,
with <sup>3</sup>J<sub>H-H</sub> = 6.6 Hz. Then, <em><strong>the signal is
further split into doublets (<sup>3</sup>J<sub>H-P</sub> = 3.3 Hz) by
P<sub>A</sub>, the closer of the two phosphorus atoms.</strong></em> A phosphorus
atom will spin-couple with protons up to three bonds away.</p>
<p><u>Exercise 9.12</u>: The signal for the two ‘H<sub>c</sub>’ protons in
isopentenyl diphosphate is reported above as a singlet integrating to
2H. Are these two protons really chemically equivalent, and, according
to what you know about proton NMR, should this signal really be a
singlet? If not, what kind of signal(s) would you expect to see? Explain
any discrepancies between what you would expect to see and the actual
reported data.</p>
<p>Now, let’s look at the <sup>13</sup>C spectrum of IPP:</p>
<p><u><sup>13</sup>C-NMR</u> (proton-decoupled)</p>
<p><strong>C<sub>1</sub></strong>: 40.7 ppm (d); <sup>2</sup>J<sub>C1-PA</sub> = 7.2 Hz</p>
<p><strong>C<sub>2</sub></strong>: 67.0 ppm (d); <sup>3</sup>J<sub>C2-PA</sub> = 4.0 Hz</p>
<p><strong>C<sub>3</sub></strong>: 147.4 ppm</p>
<p><strong>C<sub>4</sub></strong>: 114.6 ppm</p>
<p><strong>C<sub>5</sub></strong>: 24.5 ppm</p>
<img alt="../_images/image71.png" src="../_images/image71.png" />
<p>Notice that <strong>the signals for both C<sub>1</sub> and C<sub>2</sub> are
split into doublets by the magnetic field of P<sub>A</sub></strong>. Phosphorus
atoms will spin-couple with <sup>13</sup>C nuclei up to three bonds
away. Notice also that the 2-bond coupling between C<sub>1</sub> and
P<sub>A</sub> is larger than the 3-bond coupling between C<sub>2</sub>
and P<sub>A</sub> (7.2 Hz vs. 4.0 Hz). Finally, notice that we do <em>not</em>
observe 4-bond C-P coupling: C<sub>3</sub> is not spin-coupled to
P<sub>A</sub>, and P<sub>B</sub> is not coupled to any of the
<sup>13</sup>C or <sup>1</sup>H nuclei on the molecule.</p>
<p>Remember that when processing a typical <sup>13</sup>C-NMR spectrum, we
electronically ‘turn off’ spin coupling between carbons and neighboring
protons in order to simplify the spectrum (this is referred to as
‘proton decoupling’). Proton decoupling does <em><strong>not</strong></em> turn off C-P spin
coupling.</p>
<p>Because <sup>31</sup>P is NMR-active, we can also, with an NMR
spectrophotometer equipped with a phosphorus probe, <em>directly</em> observe
the phosphorus NMR signals, just as we can directly observe the signals
from protons and <sup>13</sup>C nuclei. On an NMR instrument where
protons resonate at 300 MHz and <sup>13</sup>C nuclei resonate at 75
MHz, phosphorus resonates at 32 MHz. In <sup>31</sup>P-NMR experiments,
the reference standard used to determine the 0 ppm point is usually
phosphoric acid (tetramethylsilane, the standard 0 ppm point for
<sup>1</sup>H- and <sup>13</sup>C-NMR, doesn’t have a phosphorus atom!).
The <sup>31</sup>P-NMR spectrum of isopentenyl diphosphate has, as
expected, two peaks, each of which is <em>upfield</em> of the phosphoric acid
standard (negative chemical shifts!) and split into a doublet
(<sup>2</sup>J<sub>P-P</sub> = 20 Hz) due to 2-bond coupling between the
two phosphorus nuclei.</p>
<img alt="../_images/image72.png" src="../_images/image72.png" />
<p>fig 13</p>
<p>P<sub>A</sub>: (-)11.03 ppm (d, <sup>2</sup>J<sub>P-P</sub> = 20 Hz)</p>
<p>P<sub>B</sub>: (-)7.23 ppm (d, <sup>2</sup>J<sub>P-P</sub> = 20 Hz)</p>
<p>Notice that although the C<sub>1</sub> and C<sub>2</sub> signals were
split by P<sub>A</sub> in our <sup>13</sup>C-NMR spectrum, in the
<sup>31</sup>P-NMR spectrum the converse is not true: the P<sub>A</sub>
signal is <em>not</em> split by C<sub>1</sub> or C<sub>2</sub>. Both of these
carbons are NMR-inactive <sup>12</sup>C isotope in 99 out of 100
molecules. In addition, P-H splitting is not observed in this
<sup>31</sup>P spectrum, because proton decoupling is in effect.</p>
</section>
<section id="key-concepts-to-review">
<h2>Key concepts to review<a class="headerlink" href="#key-concepts-to-review" title="Permalink to this headline">#</a></h2>
<p>All of the reactions detailed in this chapter involved the transfer of a
phosphate group - usually a phosphate, diphosphate, or AMP group - from
one molecule (the donor) to another (the acceptor). Your learning goal
for this chapter is to recognize and understand what is happening in
these phosphate group transfer reactions, and to gain a basic
understanding of the chemistry of phosphate and other phosphate groups.</p>
<p>Be sure that you can identify and provide examples of the following
terms:</p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p>inorganic phosphate</p></th>
<th class="head"><p>organic triphosphate</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>inorganic phosphate</p></td>
<td><p>phosphate (mono)ester</p></td>
</tr>
<tr class="row-odd"><td><p>inorganic pyrophosphate</p></td>
<td><p>phosphate diester</p></td>
</tr>
<tr class="row-even"><td><p>organic (mono)phosphate</p></td>
<td><p>phosphate anhydride</p></td>
</tr>
<tr class="row-odd"><td><p>organic diphosphate</p></td>
<td><p>bridging/non-bridging oxygen</p></td>
</tr>
</tbody>
</table>
<p>Also, make sure that you can recognize and use appropriately the various
abbreviations introduced in this chapter:</p>
<p>P<em>i</em> PP<em>i</em> R-OP</p>
<p>R-OPP R-OAMP ATP</p>
<p>ADP AMP</p>
<p>… in addition to the various structural abbreviations for adenosine
mono-, di-, and triphosphate.</p>
<p>You should know the approximate pKa values for phosphoric acid and an
organic monophosphate, and be able to state the approximate net charge
(to the nearest 0.5 charge unit) of these species in buffers of
different pH levels.</p>
<p>You should be able to describe, in words and pictures, the tetrahedral
<em>sp<sup>3</sup>d</em> bonding picture for the phosphorus atom of a phosphate
group. <em>Even though the geometry is not always shown in every drawing,
always keep in mind that the phosphate group is tetrahedral.</em></p>
<p>You should be able to draw resonance contributors for different
phosphate groups, identify major versus minor contributors, and explain
why some are major and some are minor. <em>Remember - charges are shared
between non-bridging oxygens, even if they are not drawn that way!</em></p>
<p><em><strong>Absolutely critical to your success with this chapter is being able
to picture and illustrate the mechanistic pattern which we refer to as a
phosphate group transfer.</strong></em></p>
<p>Though not usually included in reaction illustrations, always remember
that charge-charge interactions with magnesium ions and hydrogen bonds
to active site amino acids both serve to increase the electrophilicity
of a phosphorus atom in donor compounds such as ATP.</p>
<p>You should understand the distinctions between the three mechanistic
models for phosphate transfer reactions - concerted,
addition-elimination, and elimination-addition - and know that the
concerted model probably most closely describes biochemical reactions.</p>
<p>You should be able to identify the apical and equatorial positions in
the pentavalent transition state of a phosphate transfer reaction, and
recognize that the reaction results in inversion at the phosphorus
center.</p>
<p>You should be able to identify the α, β, and γ phosphate groups of ATP,
as well as the ribose and adenosine parts of the molecule.</p>
<p>You should be able to explain how ATP acts as a phosphate group donor,
and why such reactions are thermodynamically favorable.</p>
<p>You should be able to draw a curved-arrow mechanism for reactions in
which ATP acts as a phosphate group donor in the phosphorylation and/or
diphosphorylation of an alcohol. You should be able to predict the
result of nucleophilic attack at the α, β, or γ phosphates of ATP.</p>
<p>In general, you should be able to propose a likely mechanism for any
phosphate transfer reaction, given the starting compounds and products.</p>
<p>Given information about the existence of a covalently linked
enzyme-substrate complex in an enzyme mechanism, you should be able to
propose a likely mechanism that accounts for such an intermedia/OCBE_IIte. For
example, after being told that the phosphotyrosine phosphatase reaction
involves a phosphocysteine intermedia/OCBE_IIte, you should be able to propose a
mechanism.</p>
</section>
<section id="problems">
<h2>Problems<a class="headerlink" href="#problems" title="Permalink to this headline">#</a></h2>
<p><em>In all of the problems that follow, feel free to use appropriate
abbreviations when drawing structures. However, always be sure <strong>not</strong>
to abbreviate regions of a structure which are directly involved in
bond-breaking or bond-forming events.</em></p>
<p><strong>P9.1:</strong> Draw a likely mechanism for reaction catalyzed by shikimate
kinase (EC 2.7.1.71) in the aromatic amino acid biosynthesis pathway).
Stereochemistry of the product is not indicated in the figure below - in
your mechanism, show the stereochemistry of the product, and explain how
you are able to predict it from your knowledge of kinase reactions.</p>
<img alt="../_images/image73.png" src="../_images/image73.png" />
<p><strong>P9.2:</strong> Draw a likely mechanism for the following reaction (EC
2.7.2.3) in the gluconeogenesis pathway, and predict what compound is
indicated by the question mark.</p>
<img alt="../_images/image74.png" src="../_images/image74.png" />
<p><strong>P9.3:</strong></p>
<p>a) Draw a likely mechanism for the following reaction (EC 6.3.4.2) from
ribonucleotide biosynthesis. Hint: what is the nucleophilic group? How
could the enzyme increase it’s nucleophilicity?</p>
<img alt="../_images/image75.png" src="../_images/image75.png" />
<p>b) Draw a mechanism for the following reaction, also from
ribonucleotide biosynthesis (EC 6.3.3.1):</p>
<img alt="../_images/image76.png" src="../_images/image76.png" />
<p><strong>P9.4:</strong></p>
<p>a) The carboxylate group on the amino acid valine is activated in an
early step in the biosynthesis of the antibiotic penicillin. Predict the
product of this reaction, and draw the likely intermedia/OCBE_IIte of the
phosphate group transfer reaction.</p>
<img alt="../_images/image77.png" src="../_images/image77.png" />
<p><strong>P9.5:</strong> The reaction below is an early step in the synthesis of
tyvelose, a sugar found on the surface of some pathogenic bacteria.
Notice that CTP plays the role of the phosphate group donor in this
case, rather than ATP.</p>
<img alt="../_images/image78.png" src="../_images/image78.png" />
<p>Draw a mechanism for the reaction, and indicate the second product that
is released by the enzyme. (<em>J. Biol. Chem</em>. <strong>2005</strong>, <em>280</em>, 10774)</p>
<p><strong>P9.6:</strong> Draw the likely product of the following hypothetical
phosphate group transfer reactions. Specify which phosphate group of ATP
is the electrophile in each case.</p>
<p>a)</p>
<img alt="../_images/image79.png" src="../_images/image79.png" />
<p>b)</p>
<img alt="../_images/image80.png" src="../_images/image80.png" />
<p><strong>P9.7:</strong> The figure below illustrates an experiment in which a reaction
catalyzed by an <em>E. coli</em> enzyme was run in isotopically labeled water.</p>
<img alt="../_images/image81.png" src="../_images/image81.png" />
<p>a) The researchers concluded that the reaction was <em>not</em> a phosphate
group transfer. Explain their reasoning.</p>
<p>b) Draw the products that would be expected if the reaction actually
did proceed by a phosphate transfer mechanism (be sure to show
stereochemistry and the location of the <sup>18</sup>O atom).
(Biochemistry 2000, 39, 8603)</p>
<p><strong>P9.8:</strong> The reaction below proceeds with a <em>direct</em> attack by a water
molecule on the substrate, but the hydrolysis could be expected to
proceed through two possible mechanisms. Draw two possible mechanisms
for the reaction run in H<sub>2</sub><sup>18</sup>O. Trace the progress
of the <sup>18</sup>O ‘label’ throughout each mechanism to see where it
ends up: this should indicate to you how the two mechanisms could be
(and in fact were!) distinguished experimentally.</p>
<img alt="../_images/image82.png" src="../_images/image82.png" />
<p><strong>P9.9:</strong> Glucose-6-phosphate is dephosphorylated to glucose in the last
step of the gluconeogenesis pathway (EC 5.3.1.9). The reaction is not a
direct hydrolysis: like the phosphotyrosine phosphatase reaction we saw
in this chapter it involves formation of a phosphoenzyme intermedia/OCBE_IIte,
but in this case the enzyme residue acting as the initial phosphate
acceptor is an active site <em>histidine</em> rather than an asparate. Given
this information, propose a likely mechanism for the reaction.</p>
<p><strong>P9.10:</strong> (<em>This question assumes a basic knowledge of DNA structure
and the idea of supercoiling</em>). DNA topoisomerase enzymes catalyze the
temporary ‘nicking’ of one strand of double-stranded DNA, which allows
supercoiled DNA to ‘unwind’ before the nicked strand is re-ligated.
During the unwinding process, the 5’ end of the nicked strand is
transferred to a tyrosine in the enzyme’s active site, effectively
holding it in place while the 3’ end rotates. Overall, the
stereochemical configuration of the bridging phosphate is retained.
Propose a likely mechanism for this nicking and re-ligating process.
(<em>Biochemistry</em> <strong>2005</strong>, <em>44</em>, 11476.)</p>
<p><strong>P9.11:</strong> Pictured below is a series of phosphate group transfer steps
in the early part of isoprenoid biosynthesis in bacteria. With the
knowledge that the atoms in green are derived from ATP, predict the
structures of compounds A, B and C. (EC 2.7.7.60, EC 2.7.1.148, EC
4.6.1.12)</p>
<img alt="../_images/image83.png" src="../_images/image83.png" />
<p><strong>P9.12</strong>: The reaction below shows the synthesis of glucose-UDP, an
important intermedia/OCBE_IIte in carbohydrate biosynthesis. Notice that UTP
(instead of ATP) is the phosphate donor. Identify the by-product denoted
below by a question mark.</p>
<p><em>Molecules and Cells</em> <strong>2010</strong>, <em>29</em>, 397; E.C. 2.7.7.9</p>
<img alt="../_images/image84.png" src="../_images/image84.png" />
<p><strong>P9.13:</strong> Isomerization of 3-phosphoglycerate to 2-phosphoglycerate (EC
5.4.2.1, a reaction in glycolysis) has been shown to occur with the
participation of a phosphohistidine residue in the enzyme’s active site.
The two phosphate groups are distinguished in the figure below by color.
With this information, propose a mechanism for the reaction. (<em>J. Mol.
Bio</em>l. <strong>1999</strong>, <em>286</em>, 1507</p>
<img alt="../_images/image85.png" src="../_images/image85.png" />
<p><strong>P9.14</strong>: The gluconeogenesis (sugar-building) pathway enzyme
glucose-6-phosphatase catalyzes an <em>indirect</em> phosphate hydrolysis
reaction with a phosphohistidine intermedia/OCBE_IIte (‘indirect hydrolysis’ in
this context means that a water molecule does <em>not</em> directly attack
glucose-6-phosphate).</p>
<img alt="../_images/image86.png" src="../_images/image86.png" />
<p>Researchers wanted to confirm that the hydrolysis in this reaction is
indirect, rather than direct. It turns out that the same enzyme is also
capable of catalyzing the transfer of the phosphate group from
glucose-6-phosphate to the hydroxyl group on carbon #6 of another
glucose molecule (instead of to water, which is the natural reaction).
The enzyme-catalyzed transfer of phosphate between two glucose
substrates is reversible.</p>
<p>The researchers incubated the enzyme with labeled glucose-6-phosphate,
in which in the phosphate center was chiral (with the <em>R</em> configuration)
due to the incorporation of <sup>17</sup>O and <sup>18</sup>O isotopes.
They also included a high concentration of glucose in the reaction
mixture, which ensured that the glucose-to-glucose transfer reaction
predominated and hydrolysis (the ‘natural’ reaction) did not take place.
After allowing the reaction to reach equilibrium, they isolated the
glucose-6-phosphate and looked at the configuration of the phosphate
group.</p>
<p>Given what you have just learned about the enzyme mechanism, predict
what the researchers found in this experiment, explain your prediction,
and draw the appropriate structure(s), including stereochemistry. Assume
that the glucose-to-glucose mechanism is identical to the hydrolysis
mechanism, aside from the identity of the ultimate phosphate acceptor.
<em>Biochem J.,</em> <strong>1982</strong>*, 201,* 665*.*</p>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./OCBE_II"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="../Introduction.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">&lt;no title&gt;</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="Chapter%2010%20-%20Nucleophilic%20Addition%20to%20Carbonyl%20Groups.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter 10: Nucleophilic addition to carbonyl groups</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Tim Soderberg<br/>
  
      &copy; Copyright 2022.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>