
<!DOCTYPE html>

<html>
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="generator" content="Docutils 0.17.1: http://docutils.sourceforge.net/" />

    <title>Chapter 4: Structure determination, part I: Mass spectrometry, infrared, and UV-visible spectroscopy &#8212; Organic Chemistry with a Biological Emphasis</title>
    
  <!-- Loaded before other Sphinx assets -->
  <link href="../_static/styles/theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">
<link href="../_static/styles/pydata-sphinx-theme.css?digest=1999514e3f237ded88cf" rel="stylesheet">

    
  <link rel="stylesheet"
    href="../_static/vendor/fontawesome/5.13.0/css/all.min.css">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-solid-900.woff2">
  <link rel="preload" as="font" type="font/woff2" crossorigin
    href="../_static/vendor/fontawesome/5.13.0/webfonts/fa-brands-400.woff2">

    <link rel="stylesheet" type="text/css" href="../_static/pygments.css" />
    <link rel="stylesheet" href="../_static/styles/sphinx-book-theme.css?digest=5115cc725059bd94278eecd172e13a965bf8f5a9" type="text/css" />
    <link rel="stylesheet" type="text/css" href="../_static/togglebutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/copybutton.css" />
    <link rel="stylesheet" type="text/css" href="../_static/mystnb.css" />
    <link rel="stylesheet" type="text/css" href="../_static/sphinx-thebe.css" />
    <link rel="stylesheet" type="text/css" href="../_static/design-style.b7bb847fb20b106c3d81b95245e65545.min.css" />
    
  <!-- Pre-loaded scripts that we'll load fully later -->
  <link rel="preload" as="script" href="../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf">

    <script data-url_root="../" id="documentation_options" src="../_static/documentation_options.js"></script>
    <script src="../_static/jquery.js"></script>
    <script src="../_static/underscore.js"></script>
    <script src="../_static/doctools.js"></script>
    <script src="../_static/clipboard.min.js"></script>
    <script src="../_static/copybutton.js"></script>
    <script src="../_static/scripts/sphinx-book-theme.js?digest=9c920249402e914e316237a7dbc6769907cce411"></script>
    <script>let toggleHintShow = 'Click to show';</script>
    <script>let toggleHintHide = 'Click to hide';</script>
    <script>let toggleOpenOnPrint = 'true';</script>
    <script src="../_static/togglebutton.js"></script>
    <script>var togglebuttonSelector = '.toggle, .admonition.dropdown, .tag_hide_input div.cell_input, .tag_hide-input div.cell_input, .tag_hide_output div.cell_output, .tag_hide-output div.cell_output, .tag_hide_cell.cell, .tag_hide-cell.cell';</script>
    <script src="../_static/design-tabs.js"></script>
    <script>const THEBE_JS_URL = "https://unpkg.com/thebe@0.8.2/lib/index.js"
const thebe_selector = ".thebe,.cell"
const thebe_selector_input = "pre"
const thebe_selector_output = ".output, .cell_output"
</script>
    <script async="async" src="../_static/sphinx-thebe.js"></script>
    <link rel="index" title="Index" href="../genindex.html" />
    <link rel="search" title="Search" href="../search.html" />
    <link rel="next" title="Chapter 5: Structure determination, part II: Nuclear magnetic resonance spectroscopy" href="Chapter%205%20-%20Structure%20determination%2C%20part%20II%20-%20Nuclear%20magnetic%20resonance%20spectroscopy.html" />
    <link rel="prev" title="Chapter 3: Conformation and stereochemistry" href="Chapter%203%20-%20Conformation%20and%20stereochemistry.html" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />
    <meta name="docsearch:language" content="None">
    

    <!-- Google Analytics -->
    
  </head>
  <body data-spy="scroll" data-target="#bd-toc-nav" data-offset="60">
<!-- Checkboxes to toggle the left sidebar -->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<label class="overlay overlay-navbar" for="__navigation">
    <div class="visually-hidden">Toggle navigation sidebar</div>
</label>
<!-- Checkboxes to toggle the in-page toc -->
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">
<label class="overlay overlay-pagetoc" for="__page-toc">
    <div class="visually-hidden">Toggle in-page Table of Contents</div>
</label>
<!-- Headers at the top -->
<div class="announcement header-item noprint"></div>
<div class="header header-item noprint"></div>

    
    <div class="container-fluid" id="banner"></div>

    

    <div class="container-xl">
      <div class="row">
          
<!-- Sidebar -->
<div class="bd-sidebar noprint" id="site-navigation">
    <div class="bd-sidebar__content">
        <div class="bd-sidebar__top"><div class="navbar-brand-box">
    <a class="navbar-brand text-wrap" href="../index.html">
      
      
      
      <h1 class="site-logo" id="site-title">Organic Chemistry with a Biological Emphasis</h1>
      
    </a>
</div><form class="bd-search d-flex align-items-center" action="../search.html" method="get">
  <i class="icon fas fa-search"></i>
  <input type="search" class="form-control" name="q" id="search-input" placeholder="Search this book..." aria-label="Search this book..." autocomplete="off" >
</form><nav class="bd-links" id="bd-docs-nav" aria-label="Main">
    <div class="bd-toc-item active">
        
        <ul class="nav bd-sidenav bd-sidenav__home-link">
            <li class="toctree-l1">
                <a class="reference internal" href="../Introduction.html">
                    <no title>
                </a>
            </li>
        </ul>
        <p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Organic Chemistry with a Biological Emphasis Part 1
 </span>
</p>
<ul class="current nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%201%20-%20Introduction%20to%20organic%20structure%20and%20bonding%2C%20part%20I.html">
   Chapter 1: Introduction to organic structure and bonding, part I
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%202%20-%20Introduction%20to%20organic%20structure%20and%20bonding%2C%20part%20II.html">
   Chapter 2: Introduction to organic structure and bonding, part II
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%203%20-%20Conformation%20and%20stereochemistry.html">
   Chapter 3: Conformation and stereochemistry
  </a>
 </li>
 <li class="toctree-l1 current active">
  <a class="current reference internal" href="#">
   Chapter 4: Structure determination, part I: Mass spectrometry, infrared, and UV-visible spectroscopy
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%205%20-%20Structure%20determination%2C%20part%20II%20-%20Nuclear%20magnetic%20resonance%20spectroscopy.html">
   Chapter 5: Structure determination, part II: Nuclear magnetic resonance spectroscopy
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%206%20-%20Overview%20of%20organic%20reactivity.html">
   Chapter 6: Overview of organic reactivity
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%207%20-%20Acid-base%20reactions.html">
   Chapter 7: Acid-base reactions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Chapter%208%20-%20Nucleophilic%20substitution%20reactions.html">
   Chapter 8: Nucleophilic substitution reactions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="Tables.html">
   Table 1: Some characteristic absorption frequencies in IR spectroscopy
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Organic Chemistry with a Biological Emphasis Part 2
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../OCBE_II/Chapter%209%20-%20Phosphate%20Transfer%20Reactions.html">
   Chapter 9: Phosphate transfer reactions
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../OCBE_II/Chapter%2010%20-%20Nucleophilic%20Addition%20to%20Carbonyl%20Groups.html">
   Chapter 10: Nucleophilic addition to carbonyl groups
  </a>
 </li>
</ul>
<p aria-level="2" class="caption" role="heading">
 <span class="caption-text">
  Solutions
 </span>
</p>
<ul class="nav bd-sidenav">
 <li class="toctree-l1">
  <a class="reference internal" href="../Solutions/Solutions%20to%20in-chapter%20problems.html">
   Solutions to In-Chapter Exercises
  </a>
 </li>
 <li class="toctree-l1">
  <a class="reference internal" href="../Solutions/Solutions%20to%20end-of-chapter%20problems.html">
   Solutions to selected end-of-chapter problems
  </a>
 </li>
</ul>

    </div>
</nav></div>
        <div class="bd-sidebar__bottom">
             <!-- To handle the deprecated key -->
            
            <div class="navbar_extra_footer">
            Powered by <a href="https://jupyterbook.org">Jupyter Book</a>
            </div>
            
        </div>
    </div>
    <div id="rtd-footer-container"></div>
</div>


          


          
<!-- A tiny helper pixel to detect if we've scrolled -->
<div class="sbt-scroll-pixel-helper"></div>
<!-- Main content -->
<div class="col py-0 content-container">
    
    <div class="header-article row sticky-top noprint">
        



<div class="col py-1 d-flex header-article-main">
    <div class="header-article__left">
        
        <label for="__navigation"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="right"
title="Toggle navigation"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-bars"></i>
  </span>

</label>

        
    </div>
    <div class="header-article__right">
<button onclick="toggleFullScreen()"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="bottom"
title="Fullscreen mode"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-expand"></i>
  </span>

</button>

<div class="menu-dropdown menu-dropdown-repository-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Source repositories">
      <i class="fab fa-github"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="https://github.com/c-bun/OCBE-Textbook"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Source repository"
>
  

<span class="headerbtn__icon-container">
  <i class="fab fa-github"></i>
  </span>
<span class="headerbtn__text-container">repository</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/c-bun/OCBE-Textbook/issues/new?title=Issue%20on%20page%20%2FOCBE_I/Chapter 4 - Structure determination, part I - Mass spectrometry, infrared, and UV-visible spectroscopy.html&body=Your%20issue%20content%20here."
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Open an issue"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-lightbulb"></i>
  </span>
<span class="headerbtn__text-container">open issue</span>
</a>

      </li>
      
      <li>
        <a href="https://github.com/c-bun/OCBE-Textbook/edit/main/book/OCBE_I/Chapter 4 - Structure determination, part I - Mass spectrometry, infrared, and UV-visible spectroscopy.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Edit this page"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-pencil-alt"></i>
  </span>
<span class="headerbtn__text-container">suggest edit</span>
</a>

      </li>
      
    </ul>
  </div>
</div>

<div class="menu-dropdown menu-dropdown-download-buttons">
  <button class="headerbtn menu-dropdown__trigger"
      aria-label="Download this page">
      <i class="fas fa-download"></i>
  </button>
  <div class="menu-dropdown__content">
    <ul>
      <li>
        <a href="../_sources/OCBE_I/Chapter 4 - Structure determination, part I - Mass spectrometry, infrared, and UV-visible spectroscopy.md"
   class="headerbtn"
   data-toggle="tooltip"
data-placement="left"
title="Download source file"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file"></i>
  </span>
<span class="headerbtn__text-container">.md</span>
</a>

      </li>
      
      <li>
        
<button onclick="printPdf(this)"
  class="headerbtn"
  data-toggle="tooltip"
data-placement="left"
title="Print to PDF"
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-file-pdf"></i>
  </span>
<span class="headerbtn__text-container">.pdf</span>
</button>

      </li>
      
    </ul>
  </div>
</div>
<label for="__page-toc"
  class="headerbtn headerbtn-page-toc"
  
>
  

<span class="headerbtn__icon-container">
  <i class="fas fa-list"></i>
  </span>

</label>

    </div>
</div>

<!-- Table of contents -->
<div class="col-md-3 bd-toc show noprint">
    <div class="tocsection onthispage pt-5 pb-3">
        <i class="fas fa-list"></i> Contents
    </div>
    <nav id="bd-toc-nav" aria-label="Page">
        <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-1-mass-spectrometry">
   Section 4.1: Mass Spectrometry
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-overview-of-mass-spectrometry">
     4.1A: Overview of mass spectrometry
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-looking-at-mass-spectra">
     4.1B: Looking at mass spectra
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-gas-chromatography-mass-spectrometry">
     4.1C: Gas Chromatography - Mass Spectrometry
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-mass-spectrometry-of-biomolecules">
     4.1D: Mass spectrometry of biomolecules
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-2-introduction-to-molecular-spectroscopy">
   Section 4.2: Introduction to molecular spectroscopy
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-the-electromagnetic-spectrum">
     4.2A: The electromagnetic spectrum
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-overview-of-the-molecular-spectroscopy-experiment">
     4.2B: Overview of the molecular spectroscopy experiment
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-3-infrared-spectroscopy">
   Section 4.3: Infrared spectroscopy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-4-ultraviolet-and-visible-spectroscopy">
   Section 4.4: Ultraviolet and visible spectroscopy
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-the-electronic-transition">
     4.4A: The electronic transition
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-looking-at-uv-vis-spectra">
     4.4B: Looking at UV-vis spectra
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-applications-of-uv-spectroscopy-in-organic-and-biological-chemistry">
     4.4D: Applications of UV spectroscopy in organic and biological chemistry
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#key-concepts-for-review">
   Key concepts for review
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#problems">
   Problems
  </a>
 </li>
</ul>

    </nav>
</div>
    </div>
    <div class="article row">
        <div class="col pl-md-3 pl-lg-5 content-container">
            <!-- Table of contents that is only displayed when printing the page -->
            <div id="jb-print-docs-body" class="onlyprint">
                <h1>Chapter 4: Structure determination, part I: Mass spectrometry, infrared, and UV-visible spectroscopy</h1>
                <!-- Table of contents -->
                <div id="print-main-content">
                    <div id="jb-print-toc">
                        
                        <div>
                            <h2> Contents </h2>
                        </div>
                        <nav aria-label="Page">
                            <ul class="visible nav section-nav flex-column">
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-1-mass-spectrometry">
   Section 4.1: Mass Spectrometry
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-overview-of-mass-spectrometry">
     4.1A: Overview of mass spectrometry
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-looking-at-mass-spectra">
     4.1B: Looking at mass spectra
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#c-gas-chromatography-mass-spectrometry">
     4.1C: Gas Chromatography - Mass Spectrometry
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-mass-spectrometry-of-biomolecules">
     4.1D: Mass spectrometry of biomolecules
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-2-introduction-to-molecular-spectroscopy">
   Section 4.2: Introduction to molecular spectroscopy
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-the-electromagnetic-spectrum">
     4.2A: The electromagnetic spectrum
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-overview-of-the-molecular-spectroscopy-experiment">
     4.2B: Overview of the molecular spectroscopy experiment
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-3-infrared-spectroscopy">
   Section 4.3: Infrared spectroscopy
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#section-4-4-ultraviolet-and-visible-spectroscopy">
   Section 4.4: Ultraviolet and visible spectroscopy
  </a>
  <ul class="nav section-nav flex-column">
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#a-the-electronic-transition">
     4.4A: The electronic transition
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#b-looking-at-uv-vis-spectra">
     4.4B: Looking at UV-vis spectra
    </a>
   </li>
   <li class="toc-h3 nav-item toc-entry">
    <a class="reference internal nav-link" href="#d-applications-of-uv-spectroscopy-in-organic-and-biological-chemistry">
     4.4D: Applications of UV spectroscopy in organic and biological chemistry
    </a>
   </li>
  </ul>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#key-concepts-for-review">
   Key concepts for review
  </a>
 </li>
 <li class="toc-h2 nav-item toc-entry">
  <a class="reference internal nav-link" href="#problems">
   Problems
  </a>
 </li>
</ul>

                        </nav>
                    </div>
                </div>
            </div>
            <main id="main-content" role="main">
                
              <div>
                
  <section class="tex2jax_ignore mathjax_ignore" id="chapter-4-structure-determination-part-i-mass-spectrometry-infrared-and-uv-visible-spectroscopy">
<h1>Chapter 4: Structure determination, part I: Mass spectrometry, infrared, and UV-visible spectroscopy<a class="headerlink" href="#chapter-4-structure-determination-part-i-mass-spectrometry-infrared-and-uv-visible-spectroscopy" title="Permalink to this headline">#</a></h1>
<p><img alt="../_images/image373.jpg" src="../_images/image373.jpg" /></p>
<p>(Photo credit: https://www.flickr.com/photos/vamapaull/)</p>
<p>In the end, it was a ‘funky yellow color’ that led to the demise of
Charles Heller’s not-so-illustrious career in the world of collectable
art.</p>
<p><a class="reference external" href="https://en.wikipedia.org/wiki/William_Aiken_Walker">William Aiken
Walker</a> was a
19th-century ‘genre’ painter, known for his small scenes of
sharecroppers working the fields in the post-Civil War south. For much
of his career, he traveled extensively, throughout the southern states
but also to New York City and even as far as Cuba. He earned a decent
living wherever he went by setting up shop on the sidewalk and selling
his paintings to tourists, usually for a few dollars each. While he
never became a household name in the art world, he was prolific and
popular, and his paintings are today considered collectible, often
selling for upwards of ten thousand dollars.</p>
<p>In August, 1994, Robert Hicklin, an art gallery owner in Charleston,
South Carolina, was appraising a Walker painting brought to him by
another South Carolina art dealer named Rick Simons. Hicklin’s years of
experience with Walker paintings told him that something just wasn’t
right with this one - he was particularly bothered by one of the
pigments used, which he later described in a story in the <em>Maine Antique
Digest</em> as a ‘funky yellow color’. Reluctantly, he told Simons that it
almost certainly was a fake.</p>
<p>Hoping that Hicklin was wrong, Simons decided to submit his painting to
other experts for analysis, and eventually it ended up in the laboratory
of James Martin, whose company <em>Orion Analytical</em> specializes in
forensic materials analysis. Using a technique called infrared
spectroscopy, Martin was able to positively identify the suspicious
yellow pigment as an organic compound called <a class="reference external" href="http://www.colorrich.net/products-148731-0-item-268944.html">‘Pigment Yellow
3’</a>.</p>
<img alt="../_images/image374.png" src="../_images/image374.png" />
<p>fig 1a</p>
<p>As it turns out, Pigment Yellow 3 had not become available in the United
States until many years after William Aiken Walker died.</p>
<p>Simons had purchased his painting from a man named Robert Heller for
$9,500. When Heller approached him again to offer several more Walker
paintings for sale, Simons contacted the FBI. A few days later, with FBI
agents listening in, Simons agreed to buy two more Walker paintings.
When he received them, they were promptly analyzed and found to be fake.
Heller, who turned out to be a convicted felon, was arrested and
eventually imprisoned.</p>
<p>(You may want to check out a <a class="reference external" href="http://pubs.acs.org/doi/abs/10.1021/cen-v085n037.p028">short
article</a> from the
September 10, 2007 issue of <em>Chemical and Engineering</em> News for more
about this story).</p>
<p>In the first three chapters of this text, we have focused our efforts on
learning about the structure of organic compounds. Now that we know what
organic molecules look like, we can begin to address, in the next two
chapters, the question of <em>how</em> we get this knowledge in the first
place. How are chemists able to draw with confidence the bonding
arrangements in organic molecules, even simple ones such as acetone or
ethanol? How was James Martin at Orion Analytical able to identify the
chemical structure of the pigment compound responsible for the ‘funky
yellow color’ in the forged William Aiken Walker painting?</p>
<p>This chapter is devoted to three very important techniques used by
chemists to learn about the structures of organic molecules. First, we
will learn how <strong>mass spectrometry</strong> can provide us with information
about the mass of a molecule as well as the mass of fragments into which
the molecule has been broken. Then, we will begin our investigation of
<strong>molecular spectroscopy</strong>, which is the study of how electromagnetic
radiation at different wavelengths interacts in different ways with
molecules - and how these interactions can be quantified, analyzed, and
interpreted to gain information about molecular structure. After a brief
overview of the properties of light and the elements of a molecular
spectroscopy experiment, we will move to a discussion of <strong>infrared (IR)
spectroscopy</strong>, the key technique used in the detection of the Walker
forgery, and a way to learn about functional groups present in an
organic compound. Then, we will consider <strong>ultraviolet-visible (UV-vis)
spectroscopy</strong>, with which chemists gain information about conjugated
π-bonding systems in organic molecules, and how these conjugated systems
are the basis for the color of some organic compounds, including pigment
yellow #3. We will see how information from UV-vis spectroscopy can be
used to measure the concentration of biomolecules in solution.</p>
<p>Looking ahead, Chapter 5 will be devoted to <strong>nuclear magnetic resonance
(NMR) spectroscopy</strong>, where we use ultra-strong magnets and radio
frequency radiation to learn about the electronic environment of
individual atoms in a molecule and use this information to determine the
atom-to-atom bonding arrangement. For most organic chemists, NMR is one
of the most powerful analytical tools available in terms of the wealth
of detailed information it can provide about the structure of a
molecule.</p>
<p>In summary, the structure determination techniques we will be studying
in this chapter and the next primarily attempt to address the following
questions about an organic molecule:</p>
<p><u>Chapter 4:</u></p>
<blockquote>
<div><p>Mass spectrometry (MS): <em>What is the atomic weight of the molecule and
its common fragments?</em></p>
<p>Infrared (IR) spectroscopy: <em>what functional groups does the molecule
contain?</em></p>
<p>Ultraviolet-visible (UV-Vis) spectroscopy: <em>What is the nature of
conjugated π-bonding systems in the molecule?</em></p>
</div></blockquote>
<p><u>Chapter 5:</u></p>
<blockquote>
<div><p>Nuclear magnetic resonance spectroscopy (NMR): <em>What is the overall
bonding framework of the molecule?</em></p>
</div></blockquote>
<section id="section-4-1-mass-spectrometry">
<h2>Section 4.1: Mass Spectrometry<a class="headerlink" href="#section-4-1-mass-spectrometry" title="Permalink to this headline">#</a></h2>
<p>Mass spectrometry (MS) is a powerful analytical technique widely used by
chemists, biologists, medical researchers, and environmental and
forensic scientists, among others. With MS, we are looking at the mass
of a molecule, or of different fragments of that molecule.</p>
<section id="a-overview-of-mass-spectrometry">
<h3>4.1A: Overview of mass spectrometry<a class="headerlink" href="#a-overview-of-mass-spectrometry" title="Permalink to this headline">#</a></h3>
<p>There are many different types of MS instruments, but they all have the
same three essential components:</p>
<blockquote>
<div><p>1) First, there is an ionization source, where the molecule is given
a positive electrical charge, either by removing an electron or by
adding a proton.</p>
<p>2) Depending on the ionization method used, the ionized molecule may
or may not break apart into a population of smaller fragments</p>
<p>3) Next in line there is a mass analyzer, where the
positively-charged fragments are separated according to mass.</p>
<p>4) Finally, there is a detector, which detects and quantifies the
separated ions.</p>
</div></blockquote>
<p>One of the more common types of MS techniques used in the organic
laboratory is <strong>electron ionization</strong> (EIMS). Although biomolecules are
usually too polar for this technique, it is useful for the analysis of
intermediates and products in laboratory synthesis, and is a good
starting place to learn about MS in general. In a typical EIMS
experiment, the sample molecule is bombarded by a high-energy electron
beam, which has the effect of knocking a valence electron off of the
molecule to form a <strong>radical cation</strong>. Because a great deal of energy is
transferred by this bombardment process, the radical cation quickly
begins to break up into smaller <strong>fragments</strong>, some of which are cations
(both radical and non-radical) and some of which are neutral. A subset
of the initially formed radical cations do not fragment at all - these
are referred to as <strong>molecular ions</strong>. The neutral fragments are either
adsorbed onto the walls of the chamber or are removed by a vacuum
source. The molecular ion and all other cationic fragments are
accelerated down a curved tube by an electric field.</p>
<p><img alt="../_images/image375.png" src="../_images/image375.png" /></p>
<p>fig 26</p>
<p><a class="reference external" href="http://www.chem.ucalgary.ca/courses/351/Carey5th/Ch13/1334.gif">another
image</a></p>
<p>As they travel down the curved tube, the ions are deflected by a strong
magnetic field. Cations of different <strong>mass to charge (<em>m/z</em>) ratios</strong>
are deflected to a different extent, resulting in a sorting of ions by
mass (virtually all ions have charges of <em>z</em> = +1, so sorting by the
mass to charge ratio is essentially the same thing as sorting by mass).
A detector at the end of the curved flight tube records and quantifies
the sorted ions.</p>
</section>
<section id="b-looking-at-mass-spectra">
<h3>4.1B: Looking at mass spectra<a class="headerlink" href="#b-looking-at-mass-spectra" title="Permalink to this headline">#</a></h3>
<p>Below is typical output for an electron-ionization MS experiment (MS
data in the section is derived from the <a class="reference external" href="http://sdbs.db.aist.go.jp/sdbs/cgi-bin/cre_index.cgi">Spectral Database for Organic
Compounds</a>, a
free, web-based database of spectra.)</p>
<p><img alt="../_images/image376.png" src="../_images/image376.png" /></p>
<p>fig 27</p>
<p>The sample is acetone. On the horizontal axis is the value for <em>m/z</em> (as
we stated above, the charge <em>z</em> is almost always +1, so in practice this
is the same as mass). On the vertical axis is the relative abundance of
each ion detected. On this scale, the most abundant ion, called the
<strong>base peak</strong>, is set to 100%, and all other peaks are recorded relative
to this value. In the acetone spectrum above, the base peak is at <em>m/z</em>
= 43, representing a fragment of acetone with a mass of 43 amu. The
molecular weight of acetone is 58, so we can identify the peak at <em>m/z</em>
= 58 as that corresponding to the <strong>molecular ion peak</strong>, or <strong>parent
peak</strong>. In some mass spectra, the molecular ion peak is the most
abundant peak, and so it is also the base peak by definition.</p>
<p>The data collected in a mass spectrum - a series of m/z values, each
associated with a relative abundance value - is a unique ‘fingerprint’
of the compound being analyzed, so the spectrum of an unknown sample can
be used to search a computer database of MS spectra to come up with a
‘hit’, just as fingerprint information can be used by the police to
identify a suspect from a fingerprint database. In fact, mass
spectrometry is a powerful tool in criminal forensics.</p>
<p>An experienced chemist can gain a wealth of useful structural
information from a mass spectrum, even without access to a reference
database. The parent peak provides the molecular weight of the compound
being analyzed, which of course is a very useful piece of information.
However, much of the utility in electron-ionization MS comes from the
fact that the radical cations generated in the electron-bombardment
process tend to fragment in predictable ways. The base peak at <em>m/z</em> =
43 in the acetone spectrum, for example, is the result of the molecule
breaking apart into what is termed an acylium cation and a methyl
radical. The methyl radical fragment has a mass of 15 amu, but is not
detected because it is neutral, not a cation.</p>
<img alt="../_images/image377.png" src="../_images/image377.png" />
<p>fig 29</p>
<p>Notice that we do in fact see a peak in the spectrum at m/z = 15, which
corresponds to a methyl cation, formed in a different fragmentation
event in which the second product is a neutral (and thus undetected)
radical fragment.</p>
<p><u>Exercise 4.1</u>: Using the fragmentation patterns for acetone above
as a guide, predict the signals that you would find in the mass spectra
of:</p>
<p>a) 2-butanone; b) 3-hexanone; c) cyclopentanone.</p>
<p>Many other organic functional groups and bonding arrangements have
predictable fragmentation patterns. As a rule, the cationic fragments
that form in greatest abundance are those which are, relatively
speaking, the most stable (we will learn about carbocation stability in
chapter 8). A discussion of the mechanisms for fragmentation in mass
spectrometry is outside the scope of this book, but will be covered if
you take more advanced courses in organic or analytical chemistry.</p>
<p>Notice in the mass spectrum of acetone that there is a small peak at
<em>m/z</em> = 59: this is referred to as the <strong>M+1 peak</strong>. How can there be an
ion that has a greater mass than the molecular ion? A small fraction -
about 1.1% - of all carbon atoms in nature are actually the
<sup>13</sup>C rather than the <sup>12</sup>C isotope. The
<sup>13</sup>C isotope has an extra neutron in its nucleus, and thus is
heavier than <sup>12</sup>C by 1 mass unit. The M+1 peak corresponds to
those few acetone molecules in the sample which contained a
<sup>13</sup>C.</p>
<p>Molecules with several oxygen atoms sometimes show a small <strong>M+2 peak</strong>
(2 <em>m/z</em> units greater than the parent peak) in their mass spectra, due
to the presence of a small amount of <sup>18</sup>O (the most abundant
isotope of oxygen is <sup>16</sup>O). Because there are two abundant
isotopes of both chlorine (about 75% <sup>35</sup>Cl and 25%
<sup>37</sup>Cl) and bromine (about 50% <sup>79</sup>Br and 50%
<sup>81</sup>Br), chlorinated and brominated compounds have very large
and recognizable M+2 peaks. Fragments containing both isotopes of
bromine can be seen in the mass spectrum of bromoethane: notice that the
equal abundance of the two isotopes is reflected in the equal height of
their corresponding peaks.</p>
<p><img alt="../_images/image378.png" src="../_images/image378.png" /></p>
<p>fig 28</p>
<p>The spectrum for chloroethane shows a parent peak at <em>m/z</em> = 64, and an
M+2 peak at 1/3 the height of the parent peak. These characteristic M+2
patterns are useful for identifying the presence of bromine or chlorine
in a sample.</p>
<p><u>Exercise 4.2:</u> Predict some signals that you would expect to see
in a mass spectrum of 2-chloropropane.</p>
<p><u>Exercise 4.3</u>: The mass spectrum of an aldehyde shows a parent
peak at <em>m/z</em> = 58 and a base peak at <em>m/z</em> = 29. Propose a structure,
and identify the two species whose <em>m/z</em> values were listed. (</p>
</section>
<section id="c-gas-chromatography-mass-spectrometry">
<h3>4.1C: Gas Chromatography - Mass Spectrometry<a class="headerlink" href="#c-gas-chromatography-mass-spectrometry" title="Permalink to this headline">#</a></h3>
<p>Quite often, mass spectrometry is used in conjunction with a separation
technique called gas chromatography (GC). The combined GC-MS procedure
is very useful when dealing with a sample that is a mixture of two or
more different compounds, because the various compounds are separated
from one another before being subjected individually to MS analysis. We
will not go into the details of gas chromatography here, although if you
are taking an organic laboratory course you might well get a chance to
try your hand at GC, and you will almost certainly be exposed to the
conceptually analogous techniques of thin layer and column
chromatography. Suffice it to say that in GC, a very small amount of a
liquid sample is vaporized, injected into a long, coiled capillary
column, and pushed though the column by an inert gas, usually helium.
Along the way, different compounds in the sample stick to the walls of
the capillary column to different extents, and thus travel at different
speeds and emerge separately from the end of the column. In GC-MS, each
separated compound is sent directly from the end of the GC column into
the MS instrument, so in the end we get a separate mass spectrum for
each of the components in the original mixed sample.</p>
<p><a class="reference external" href="https://www.researchgate.net/profile/Gaojin_Lyu/publication/221927526/figure/fig1/AS:304730011127825&#64;1449664704989/Fig-1-Schematic-of-a-GC-MS-system.png">Schematic of a GC-MS
instrument</a></p>
<p>Ideally, this data can be used to search an MS database and identify
each compound in the sample.</p>
<p>The extremely high sensitivity of modern GC-MS instrumentation makes it
possible to detect and identify very small trace amounts of organic
compounds. GC-MS is being used increasingly by environmental chemists to
detect the presence of harmful organic contaminants in food and water
samples.</p>
</section>
<section id="d-mass-spectrometry-of-biomolecules">
<h3>4.1D: Mass spectrometry of biomolecules<a class="headerlink" href="#d-mass-spectrometry-of-biomolecules" title="Permalink to this headline">#</a></h3>
<p><em>Methods Enzymol</em>. <strong>2005</strong>, <em>402</em>, 3.</p>
<p>As was mentioned earlier, electron ionization mass spectrometry is
generally not very useful for analyzing biomolecules: their high
polarity makes it difficult to get them into the vapor phase, the first
step in EIMS. Mass spectrometry of biomolecules has undergone a
revolution over the past few decades, with many new ionization and
separation techniques being developed. Generally, the strategy for
biomolecule analysis involves <strong>soft ionization</strong>, in which much less
energy (compared to techniques such as EIMS) is imparted to the molecule
being analyzed during the ionization process. Usually, soft ionization
involves adding protons rather than removing electrons: the cations
formed in this way are significantly less energetic than the radical
cations formed by removal of an electron. The result of soft ionization
is that little or no fragmentation occurs, so the mass being measured is
that of an intact molecule. Typically, large biomolecules are digested
into smaller pieces using chemical or enzymatic methods, then their
masses determined by ‘soft’ MS.</p>
<p>New developments in soft ionization MS technology have made it easier to
detect and identify proteins that are present in very small quantities
in biological samples. In <strong>electrospray ionization</strong> (ESI), the protein
sample, in solution, is sprayed into a tube and the molecules are
induced by an electric field to pick up extra protons from the solvent.
Another common ‘soft ionization’ method is ‘matrix-assisted laser
desorption ionization’ (<strong>MALDI</strong>). Here, the protein sample is adsorbed
onto a solid matrix, and protonation is achieved with a laser.</p>
<p>Typically, both electrospray ionization and MALDI are used in
conjunction with a time-of-flight (TOF) mass analyzer component.</p>
<img alt="../_images/image379.png" src="../_images/image379.png" />
<p>fig 30</p>
<p><a class="reference external" href="http://1.bp.blogspot.com/-DFjr9HnXktE/Ugvj7PQlmvI/AAAAAAAABtc/w383gMpx5mA/s1600/maldi-tof-principle.jpg">another schematic of
MALDI-TOF</a></p>
<p>The proteins are accelerated by an electrode through a column, and
separation is achieved because lighter ions travel at greater velocity
than heavier ions with the same overall charge. In this way, the many
proteins in a complex biological sample (such as blood plasma, urine,
etc.) can be separated and their individual masses determined very
accurately. Modern protein MS is extremely sensitive – recently,
scientists were even able to detect the presence of <em>Tyrannosaurus rex</em>
protein in a fossilized skeleton! (<a class="reference external" href="http://science.sciencemag.org/content/316/5822/277"><em>Science</em> <strong>2007</strong>, <em>316</em>,
277</a>).</p>
<p>Soft ionization mass spectrometry has become in recent years an
increasingly important tool in the field of
<a class="reference external" href="https://www.ebi.ac.uk/training/online/course/proteomics-introduction-ebi-resources/what-proteomics"><strong>proteomics</strong></a>.
Traditionally, protein biochemists tend to study the structure and
function of individual proteins. Proteomics researchers, in contrast,
want to learn more about how large numbers of proteins in a living
system interact with each other, and how they respond to changes in the
state of the organism. One important subfield of proteomics is the
search for protein ‘biomarkers’ for human disease: in other words,
proteins which are present in greater quantities in the tissues of a
sick person than in a healthy person. Detection in a healthy person of a
known biomarker for a disease such as diabetes or cancer could provide
doctors with an early warning that the patient may be especially
susceptible to the disease, so that preventive measures could be taken
to prevent or delay onset.</p>
<p>In a 2005 study, MALDI-TOF mass spectrometry was used to compare fluid
samples from lung transplant recipients who had suffered from tissue
rejection to samples from recipients who had not suffered rejection.
Three peptides (short proteins) were found to be present at elevated
levels specifically in the tissue rejection samples. It is hoped that
these peptides might serve as biomarkers to identify patients who are at
increased risk of rejecting their transplanted lungs. (<a class="reference external" href="http://onlinelibrary.wiley.com/doi/10.1002/pmic.200401036/abstract"><em>Proteomics</em>
<strong>2005</strong>, <em>5</em>,
1705</a>).
More recently, MALDI-TOF MS was used to identify protein biomarkers for
Alzheimer’s disease in the blood, which could lead to an early diagnosis
method for Alzheimer’s which is much less expensive than brain scans,
the only detection method currently available (<a class="reference external" href="https://www.nature.com/articles/nature25456"><em>Nature</em> <strong>2018</strong>,
<em>554</em>, 249</a>).</p>
</section>
</section>
<section id="section-4-2-introduction-to-molecular-spectroscopy">
<h2>Section 4.2: Introduction to molecular spectroscopy<a class="headerlink" href="#section-4-2-introduction-to-molecular-spectroscopy" title="Permalink to this headline">#</a></h2>
<section id="a-the-electromagnetic-spectrum">
<h3>4.2A: The electromagnetic spectrum<a class="headerlink" href="#a-the-electromagnetic-spectrum" title="Permalink to this headline">#</a></h3>
<p>Electromagnetic radiation, as you may recall from a previous chemistry
or physics class, is composed of oscillating electrical and magnetic
fields. Visible light is electromagnetic radiation. So are the gamma
rays that are emitted by spent nuclear fuel, the x-rays that a doctor
uses to visualize your bones, the ultraviolet light that causes a
painful sunburn when you forget to apply sun block, the infrared light
that the army uses in night-vision goggles, the microwaves you use to
heat up your frozen burritos, and the radio-frequency waves of your cell
phone signal.</p>
<p>While the speed of a wave in the ocean can vary, the speed of
electromagnetic waves – commonly referred to as the speed of light – is
a constant, approximately 300 million meters per second, whether we are
talking about gamma radiation, visible light, or FM radio waves.
Electromagnetic radiation is defined by its <strong>wavelength</strong>, which is the
distance between one wave crest to the next.</p>
<img alt="../_images/image380.png" src="../_images/image380.png" />
<p>Because electromagnetic radiation travels at a constant speed, each
wavelength corresponds to a given <strong>frequency</strong>, which is the number of
times per second that a crest passes a given point. Longer waves have
lower frequencies, and shorter waves have higher frequencies. Frequency
is commonly reported in hertz (Hz), meaning ‘cycles per second’, or
‘waves per second’. The equivalent standard unit for frequency is
s<sup>-1</sup>.</p>
<p>When talking about electromagnetic waves, we can refer either to
wavelength or to frequency - the two values are inversely proportional:</p>
<p>equation 4.1 λυ = <em>c</em></p>
<p>where λ (Greek ‘<em>lambda</em>’) is wavelength, υ (Greek ‘<em>nu’</em>) is frequency
in s<sup>-1</sup> (or Hz), and <em>c</em> is the speed of light, a constant
value of 3.0 x 10<sup>8</sup> m·s<sup>-1</sup>.</p>
<p>Electromagnetic radiation transmits energy in discreet quantum
‘packages’ called <strong>photons</strong>. Shorter wavelengths (and higher
frequencies) correspond to higher energy.</p>
<p>High energy radiation, such as gamma radiation and x-rays, is composed
of very short waves – as short as 10<sup>-16</sup> m. Longer wavelengths
are far less energetic, and thus are less dangerous to living things.
Visible light waves are in the range of 400 – 700 nm (nanometers, or
10<sup>-9</sup> m), while radio waves can be several hundred meters in
length.</p>
<p>The energy of a particular wavelength of electromagnetic radiation can
be expressed as:</p>
<p>equation 4.2 E = <em>hc</em>/λ = <em>h</em>ν</p>
<p>where E is energy in kJ/mol of photons and <em>h</em> is 3.99 x
10<sup>-13</sup> kJ·s·mol<sup>-1</sup>, a number known as <strong>Planck’s
constant</strong>.</p>
<p>Using equations 4.1 and 4.2, we can calculate, for example, that visible
red light with a wavelength of 700 nm has a frequency of 4.29 x
10<sup>14</sup> Hz, and an energy of 171 kJ per mole of photons.</p>
<p>The full range of electromagnetic radiation wavelengths is referred to
as the <strong>electromagnetic spectrum</strong>.</p>
<img alt="../_images/image381.png" src="../_images/image381.png" />
<p>fig 1</p>
<p><a class="reference external" href="https://imagine.gsfc.nasa.gov/Images/science/EM_spectrum_compare_level1_lg.jpg">another
image</a></p>
<p>Notice in the figure above that visible light takes up just a narrow
band of the full spectrum. White light from the sun or a light bulb is a
mixture of all of the visible wavelengths plus some UV. You see the
visible region of the electromagnetic spectrum divided into its
different wavelengths every time you see a rainbow: violet light has the
shortest wavelength, and red light has the longest.</p>
<p><u>Exercise 4.4</u>: Visible light has a wavelength range of about
400-700 nm. What is the corresponding frequency range? What is the
corresponding energy range, in kJ/mol of photons?</p>
</section>
<section id="b-overview-of-the-molecular-spectroscopy-experiment">
<h3>4.2B: Overview of the molecular spectroscopy experiment<a class="headerlink" href="#b-overview-of-the-molecular-spectroscopy-experiment" title="Permalink to this headline">#</a></h3>
<p>In a molecular spectroscopy experiment, electromagnetic radiation of a
specified range of wavelengths is allowed to pass through a sample
containing a compound of interest. The sample molecules absorb energy
from some of the wavelengths, and as a result jump from a low energy
‘ground state’ to some higher energy ‘excited state’. Other wavelengths
are <em>not</em> absorbed by the sample molecule, so they pass on through. A
detector records which wavelengths were absorbed, and to what extent
they were absorbed.</p>
<p><a class="reference external" href="https://upload.wikimedia.org/wikipedia/commons/thumb/f/f2/Spectroscopy_overview.svg/1200px-Spectroscopy_overview.svg.png">General schematic of molecular
spectroscopy</a></p>
<p>As we will see in this chapter, we can learn a lot about the structure
of an organic molecule by quantifying how it absorbs (or does not
absorb) different wavelengths in the electromagnetic spectrum. Three of
the most useful types of molecular spectroscopy for organic chemists
involve absorption of radiation in the infrared, ultraviolet/visible,
and radio regions of the electromagnetic spectrum. We will focus first
on infrared spectroscopy.</p>
</section>
</section>
<section id="section-4-3-infrared-spectroscopy">
<h2>Section 4.3: Infrared spectroscopy<a class="headerlink" href="#section-4-3-infrared-spectroscopy" title="Permalink to this headline">#</a></h2>
<p>Covalent bonds in organic molecules are not rigid sticks – rather, they
behave more like springs. At room temperature, organic molecules are
always in motion, as their bonds stretch, bend, and twist. These complex
vibrations can be broken down mathematically into individual
<strong>vibrational modes</strong>, a few of which are illustrated below.</p>
<p><img alt="../_images/image382.png" src="../_images/image382.png" /></p>
<p>fig 3</p>
<p><a class="reference external" href="http://www.chemtube3d.com/vibrationsH2O.htm">interactive: vibrational modes of
water</a></p>
<p>Each vibrational mode for any bond in an organic molecule has a
characteristic ground state frequency of vibration, which will be
somewhere between 10<sup>13</sup> and 10<sup>14</sup> vibrations per
second. This range of frequencies corresponds to the infrared region of
the electromagnetic spectrum.</p>
<p>If a molecule is exposed to electromagnetic radiation that matches the
frequency of one of its vibrational modes, it will absorb energy from
the radiation and jump to a higher vibrational energy state - the
<em>amplitude</em> of the vibration will increase, but the vibrational
<em>frequency</em> will remain the same. The difference in energy between the
two vibrational states is equal to the energy associated with the
wavelength of radiation that was absorbed.</p>
<p>Let’s take 2-hexanone as an example. Picture the carbonyl bond of the
ketone group as a spring. This spring is constantly bouncing back and
forth, stretching and compressing, pushing the carbon and oxygen atoms
further apart and then pulling them together. This is the <strong>stretching
mode</strong> of the carbonyl bond. In the space of one second, the spring
‘bounces’ back and forth 5.15 x 10<sup>13</sup> times - in other words,
its stretching frequency is 5.15 x 10<sup>13</sup> Hz (remember that Hz
means ‘per second’).</p>
<p><a class="reference external" href="https://www.youtube.com/watch?v=ho224YwTsG8">animation of carbonyl stretching in several organic
molecules</a></p>
<p>If our ketone sample is irradiated with infrared light, the carbonyl
bond will absorb light of a frequency matching its stretching frequency
of 5.15 x 10<sup>13</sup> Hz, which by equations 4.1 and 4.2 corresponds
to a wavelength of λ = 5.83 x 10<sup>-6</sup> m and an energy of 20.5
kJ/mol. When the carbonyl bond absorbs this energy, it jumps up to its
excited vibrational state.</p>
<img alt="../_images/image383.png" src="../_images/image383.png" />
<p>fig 4</p>
<p>The molecule does not remain in the excited vibrational state for very
long, but quickly releases energy to the surrounding environment in form
of heat, and returns to the ground state.</p>
<p>In an <a class="reference external" href="http://www.chemhume.co.uk/ASCHEM/Unit%202/Ch10%20Alkenes/infrared_spectrometer.jpg"><strong>infrared
spectrophotometer</strong></a>,
light in the infrared region (frequencies ranging from about
10<sup>13</sup> to 10<sup>14</sup> Hz) is passed though our sample of
2-hexanone. Most frequencies pass right through the sample. The 5.15 x
10<sup>13</sup> Hz carbonyl stretching frequency, however, is absorbed
by the 2-hexanone sample, and a detector in the instrument records that
the intensity of this frequency, after having passed through the sample,
is something less than 100% of its initial value.</p>
<p>The vibrations of a 2-hexanone molecule are not, of course, limited to
the simple stretching of the carbonyl bond. The various carbon-carbon
bonds also stretch and bend, as do the carbon-hydrogen bonds, and all of
these vibrational modes also absorb different frequencies of infrared
light.</p>
<p>The power of infrared spectroscopy arises from the observation that <em>the
bonds characterizing different functional groups have different
characteristic absorption frequencies</em>. The carbonyl bond in a ketone,
as we saw with our 2-hexanone example, typically absorbs in the range of
5.11 - 5.18 x 10<sup>13</sup> Hz, depending on the molecule. The
carbon-carbon triple bond of an alkyne, on the other hand, absorbs in
the range of 6.30 - 6.80 x 10<sup>13</sup> Hz. The technique is
therefore very useful as a means of identifying which functional groups
are present in a molecule of interest. If we pass infrared light through
an unknown sample and find that it absorbs in the carbonyl frequency
range but not in the alkyne range, we can infer that the molecule
contains a carbonyl group but not an alkyne.</p>
<p>Some bonds absorb infrared light more strongly than others, and some
bonds do not absorb at all. <em>In order for a vibrational mode to absorb
infrared light, it must result in a periodic (repeating) change in the
dipole moment of the molecule</em>. Such vibrations are said to be
<strong>infrared active</strong>. In general, the greater the polarity of the bond,
the stronger its IR absorption. The carbonyl bond is very polar, and
therefore absorbs very strongly. The carbon-carbon triple bond in most
alkynes, in contrast, is much less polar, and thus a stretching
vibration does not result in a large change in the overall dipole moment
of the molecule. We will see below that IR signals for carbonyl bonds
are very strong, but the signals for carbon-carbon double and triple
bonds are relatively weak.</p>
<p>Some kinds of vibrations are <strong>infrared inactive</strong>. The stretching
vibrations of completely symmetrical double and triple bonds, for
example, do not result in a change in dipole moment, and therefore do
not result in any absorption of light (but other bonds and vibrational
modes in these molecules <em>do</em> absorb IR light).</p>
<img alt="../_images/image384.png" src="../_images/image384.png" />
<p>fig 6</p>
<p>Now, let’s look at some actual output from IR spectroscopy experiments.
Below is the IR spectrum for 2-hexanone.</p>
<p><img alt="../_images/image385.png" src="../_images/image385.png" /></p>
<p>fig 7</p>
<p>There are a number of things that need to be explained in order for you
to understand what it is that we are looking at. On the horizontal axis
we see IR wavelengths expressed in terms of a unit called <strong>wavenumber</strong>
(cm<sup>-1</sup>), which tells us how many waves fit into one
centimeter.</p>
<p>equation 4.3: wavenumber (in cm<sup>-1</sup>) = 1/100λ ν/100<em>c</em></p>
<p>where λ is the wavelength in meters, v is frequency in s<sup>-1</sup>,
and <em>c</em> is the speed of light, 3.0 x 10<sup>8</sup> m/s. Wavenumber
units are typically used in IR spectra, but you may also occasionally
see micrometer (μm) units) on the horizontal axis.</p>
<p>On the vertical axis we see ‘% transmittance’, which tells us how
strongly light was absorbed at each frequency (100% transmittance, at
the high end of the axis, means no absorption occurred at that
frequency). The solid line traces the values of % transmittance for
every wavelength passed through the sample – the downward-pointing
<strong>absorbance bands</strong> show regions of strong absorption.</p>
<p>Exercise 4.5: Express the wavenumber value of 3000 cm<sup>-1</sup> in
terms of wavelength (in meter units) frequency (in Hz), and associated
energy (in kJ/mol).</p>
<p>The key absorption band in this spectrum is that from the carbonyl
double bond, at</p>
<p>1716 cm<sup>-1</sup>, corresponding to a wavelength of 5.86 μm, a
frequency of 5.15 x 10<sup>13</sup> Hz, and a ∆E value of 20.5 kJ/mol.
Notice how strong this band is, relative to the others on the spectrum:
<em>a strong absorbance band in the 1650-1750 cm<sup>-1</sup> region tells
us that a carbonyl group is present</em>. Within that range, carboxylic
acids, esters, ketones, and aldehydes tend to absorb in the shorter
wavelength end (1700-1750 cm-1), while conjugated unsaturated ketones
and amides tend to absorb on the longer wavelength end (1650-1700
cm<sup>-1</sup>).</p>
<p>The jagged band at approximately 2900-3000 cm<sup>-1</sup> is
characteristic of the stretching modes of <em>sp<sup>3</sup></em>-hybridized
carbon-hydrogen bonds. This band is not terribly useful, as just about
every organic molecule that you will have occasion to analyze has these
bonds. Nevertheless, it can serve as a familiar reference point to
orient yourself in a spectrum.</p>
<p>You will notice that there are many additional band in this spectrum in
the longer-wavelength 400 -1400 cm<sup>-1</sup> region. This part of the
spectrum is called the <strong>fingerprint region</strong>. While it is usually very
difficult to pick out anything specific to a functional group from this
region, the fingerprint region nevertheless contains valuable
information. Just like a human fingerprint, the pattern of absorbance
bands in the fingerprint region is unique to every molecule, meaning
that the data from an unknown sample can be compared to a database of IR
spectra of known standards in order to make a positive identification.
In our introductory story about the forged paintings, it was the IR
fingerprint region of the suspicious yellow paint that allowed for its
identification as a pigment that could not possibly have been used by
the purported artist, William Aiken Walker. (Recall from earlier in this
chapter that a mass spectrum of a molecule can also be thought of as a
kind of ‘fingerprint’ by which the molecule can be identified.)</p>
<p>Now, let’s take a look at the IR spectrum for 1-hexanol.</p>
<img alt="../_images/image386.png" src="../_images/image386.png" />
<p>fig 8</p>
<p>As expected, there is no carbonyl band at around 1700 cm<sup>-1</sup>,
but we can see a very broad ‘mountain’ centered at about 3400
cm<sup>-1</sup>. This signal is characteristic of the O-H stretching
mode of alcohols, and tells us that our sample has an alcohol group. The
breadth of this signal is a consequence of hydrogen bonding between
molecules.</p>
<p>The spectrum for 1-octene shows two bands that are characteristic of
alkenes: the one at 1642 cm<sup>-1</sup> is due to stretching of the
carbon-carbon double bond, and the one at 3079 cm-1 is due to stretching
of the σ bond between the <em>sp<sup>2</sup></em>-hybridized alkene carbons and
their attached hydrogens.</p>
<p><img alt="../_images/image387.png" src="../_images/image387.png" /></p>
<p>fig 10</p>
<p><a class="reference external" href="https://image.slidesharecdn.com/theoryofirspectroscopy-160622080111/95/theory-of-ir-spectroscopy-41-638.jpg?cb=1466582588">IR spectrum of
(Z)-2-octene</a></p>
<p>Alkynes have characteristic weak but sharp IR absorbance bands in the
range of 2100-2250 cm<sup>-1</sup> due to stretching of the
carbon-carbon triple bond, and terminal alkynes can be identified by
their absorbance at about 3300 cm<sup>-1</sup>, due to stretching of the
bond between the <em>sp</em>-hybridized carbon and the terminal hydrogen.)</p>
<p><a class="reference external" href="http://orgchemboulder.com/Spectroscopy/irtutor/images/hexyne.gif">IR spectrum of
1-hexyne</a></p>
<p><u>Exercise 4.6</u>: Explain how you could use the C-C and C-H
stretching frequencies in IR spectra to distinguish between four
constitutional isomers: 1,2-dimethylcyclohexene, 1,3-octadiene,
3-octyne, and 1-octyne.</p>
<p>Other functional groups can be identified IR, but we will confine our
discussion here to a small set of the most easily identifiable groups.
These are summarized below.</p>
<p><strong>Characteristic IR absorbances</strong></p>
<table class="colwidths-auto table">
<thead>
<tr class="row-odd"><th class="head"><p><strong>Functional group</strong></p></th>
<th class="head"><p><strong>Characteristic IR absorbance(s) (cm<sup>-1</sup>)</strong></p></th>
<th class="head"><p><strong>Source of signal</strong></p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>carbonyl</p></td>
<td><p>1650-1750 (strong)</p></td>
<td><p>C=O stretching</p></td>
</tr>
<tr class="row-odd"><td><p>alcohol</p></td>
<td><p>3200 - 3600 (broad)</p></td>
<td><p>O-H stretching</p></td>
</tr>
<tr class="row-even"><td><p>carboxylic acid</p></td>
<td><p>1700-1725 (strong)</p></td>
<td><p>C=O stretching</p></td>
</tr>
<tr class="row-odd"><td><p></p></td>
<td><p>2500-3000 (broad)</p></td>
<td><p>O-H stretching</p></td>
</tr>
<tr class="row-even"><td><p>alkene</p></td>
<td><p>1620 - 1680 (weak)</p></td>
<td><p>C=C stretching</p></td>
</tr>
<tr class="row-odd"><td><p></p></td>
<td><p>3020 - 3080</p></td>
<td><p>vinylic C-H stretching</p></td>
</tr>
<tr class="row-even"><td><p>alkyne</p></td>
<td><p>1620 - 1680 (weak)</p></td>
<td><p>triple bond stretching</p></td>
</tr>
<tr class="row-odd"><td><p></p></td>
<td><p>3250-3350</p></td>
<td><p>terminal C-H stretching</p></td>
</tr>
</tbody>
</table>
<p>A more detailed summary is provided
<a class="reference external" href="http://classes.kvcc.edu/chm220/Images/table1a_IR.jpg">here</a> and in
Table 1 at the end of this book.</p>
<p>An IR spectrum usually does not provide enough information for us to
figure out the complete structure of a molecule. In conjunction with
other analytical methods, however, IR spectroscopy can prove to be a
very valuable tool, given the information it provides about the presence
or absence of key functional groups. IR can also be a quick and
convenient way for a chemist to check to see if a reaction has proceeded
as planned. If we were to run a reaction in which we wished to convert
cyclohexanone to cyclohexanol, for example, a quick comparison of the IR
spectra of starting compound and product would tell us if we had
successfully converted the ketone group to an alcohol.</p>
<p><a class="reference external" href="https://www2.chemistry.msu.edu/faculty/reusch/virttxtjml/spectrpy/Images/propacid.gif">IR spectrum of a carboxylic acid (propionic
acid)</a></p>
<p><a class="reference external" href="http://1.bp.blogspot.com/-UJ061-CyE6Y/UMqqoaJk9OI/AAAAAAAADLs/BPDj6bVJFS4/s1600/Introduction_to_Spectroscopy+(1).pdf+-+SumatraPDF_2012-12-13_21-26-52.png">IR spectrum of an amide
(N-methylaceetamide)</a></p>
<p><a class="reference external" href="http://orgchemboulder.com/Spectroscopy/irtutor/images/dietham.gif">IR spectrum of
diethylamine</a></p>
<p><u>Exercise 4.7</u>: Using the online <a class="reference external" href="http://sdbs.db.aist.go.jp/sdbs/cgi-bin/cre_index.cgi">Spectral Database for Organic
Compounds</a>, look
up IR spectra for the following compounds, and identify absorbance bands
corresponding to those listed in the table above. List actual
frequencies for each signal to the nearest cm<sup>-1</sup> unit, using
the information in tables provided on the site.</p>
<p>a) 1-methylcyclohexanol</p>
<p>b) 4-methylcyclohexene</p>
<p>c) 1-hexyne</p>
<p>d) 2-hexyne</p>
<p>e) 3-hexyne-2,5-diol</p>
<p><u>Exercise 4.8</u>: A carbon-carbon single bond absorbs in the
fingerprint region, and we have already seen the characteristic
absorption wavelengths of carbon-carbon double and triple bonds.
Rationalize the trend in wavelengths. (<em>Hint</em> - remember, we are
thinking of bonds as springs, and looking at the frequency at which they
‘bounce’).</p>
</section>
<section id="section-4-4-ultraviolet-and-visible-spectroscopy">
<h2>Section 4.4: Ultraviolet and visible spectroscopy<a class="headerlink" href="#section-4-4-ultraviolet-and-visible-spectroscopy" title="Permalink to this headline">#</a></h2>
<p>Ultraviolet and visible (UV-Vis) spectroscopy provides us with
information about aromatic groups and other conjugated π systems in
organic compounds. In looking at IR spectroscopy, we saw how interaction
with infrared light causes molecules to undergo vibrational transitions.
The shorter wavelength, higher energy radiation in the UV (200-400 nm)
and visible (400-700 nm) range of the electromagnetic spectrum causes
organic molecules with conjugated π bonds to undergo <strong>electronic
transitions</strong>.</p>
<section id="a-the-electronic-transition">
<h3>4.4A: The electronic transition<a class="headerlink" href="#a-the-electronic-transition" title="Permalink to this headline">#</a></h3>
<p>Recall from chapter 2 that molecular orbital (MO) theory can be a useful
way to think about covalent bonding in molecules containing conjugated
double bonds and aromatic rings. Keep in mind that molecular orbitals
result from the mathematical combination of two or more atomic orbitals,
and describe a region in space around a <em>group of atoms</em> (rather than a
single atom) in which electrons are likely to be found. We’ll see here
that MO theory is indispensible to a discussion of ultraviolet and
visible spectroscopy.</p>
<p>A <strong>chromophore</strong> is any molecule or part of a molecule that has the
property of absorbing light in the ultraviolet or visible region of the
spectrum. 1,3-butadiene is a chromophore. Recall from section 2.2B that
the conjugated π bonds in1,3-butadiene can be pictured as four parallel,
overlapping <em>p</em> orbitals sharing four electrons:</p>
<p><img alt="../_images/image388.png" src="../_images/image388.png" /></p>
<p>fig 11</p>
<p>Using the language of MO theory, we can say that the four <em>p</em> atomic
orbitals combine to form four π molecular orbitals. The two lower energy
orbitals π<sub>1</sub> and π<sub>2</sub> are bonding orbitals, while the
two higher energy orbitals π<sub>3</sub>* and π<sub>4</sub>* are
antibonding orbitals.</p>
<p><img alt="../_images/image389.png" src="../_images/image389.png" /></p>
<p>fig 12</p>
<p>In the electronic ground state, all four π electrons in the conjugated
system reside in the two bonding orbitals. A **π **→ <strong>π* transition</strong>
occurs when an electron in π<sub>2</sub>, the <strong>highest occupied
molecular orbital (HOMO)</strong> jumps up to π<sub>3</sub>*, the <strong>lowest
unoccupied molecular orbital (LUMO)</strong>. The gap between these two
molecule orbitals is often called the <strong>HOMO-LUMO energy gap</strong>.</p>
<p>For 1,3-butadiene, the energy difference in the HOMO-LUMO gap is 552
kJ/mol. This energy corresponds to light with a wavelength of 217 nm,
which is in the ultraviolet region of the spectrum. When a sample of
1,3-butadiene is irradiated with UV radiation, the molecule absorbs the
light at 217 nm as an electron undergoes a π-π* transition.</p>
<p><img alt="../_images/image390.png" src="../_images/image390.png" /></p>
<p>fig 13</p>
<p>The absorbance due to the π → π* transition in 1,3,5-hexatriene occurs
at 258 nm, corresponding to a HOMO-LUMO gap of 464 kJ/mol.</p>
<p><img alt="../_images/image391.png" src="../_images/image391.png" /></p>
<p>fig 14</p>
<p>Notice what happened here: when the conjugated π system was extended by
two carbons, the HOMO-LUMO energy gap shrunk from 552 to 464 kJ/mol. A
lower energy transition corresponds to longer wavelength absorbance
(look again at equation 4.2 in the previous section). In general:</p>
<p><em>As conjugated π systems become more extended,</em></p>
<p><em>a) the HOMO-LUMO gap shrinks, and</em></p>
<p><em>b) the wavelength of absorbed light becomes longer</em>.</p>
<p>In molecules with very large, extended conjugated π systems, the π → π*
energy gap becomes so small that absorption occurs in the visible rather
than the UV region of the electromagnetic spectrum.</p>
<p>The visible region of the electromagnetic spectrum:</p>
<p><img alt="../_images/image392.png" src="../_images/image392.png" /></p>
<p>fig 14a</p>
<p>β-carotene, with its system of 11 conjugated double bonds, absorbs light
in a broad band between approximately 420-550 nm, with a λ<sub>max</sub>
at 470 nm, spanning the blue and green regions of the visible spectrum.
Because blue/green wavelengths are absorbed while those in the red and
yellow regions are able to pass through the sample, β-carotene appears
to our eyes as orange. β-carotene is abundant in carrots, and is the
compound mainly responsible for their orange color.</p>
<p><img alt="../_images/image393.png" src="../_images/image393.png" /></p>
<p>fig 15</p>
<p><u>Exercise 4.9</u>: What is the energy of the photons (in kJ/mol) of
light with wavelength of 470 nm, the λ<sub>max</sub> of β-carotene?</p>
<p><u>Exercise 4.10</u>: Which of the following molecules would you expect
absorb at a longer wavelength in the UV region of the electromagnetic
spectrum? Explain your answer.</p>
<blockquote>
<div><p><img alt="../_images/image394.png" src="../_images/image394.png" /></p>
</div></blockquote>
<p>fig19a</p>
<p><strong>Protecting yourself from sunburn</strong></p>
<p>Human skin can be damaged by exposure to ultraviolet light from the sun.
We naturally produce a pigment, called melanin, which protects the skin
by absorbing much of the ultraviolet radiation. Melanin is a complex
polymer, two of the most common monomers units of which are shown below.</p>
<p><img alt="../_images/image395.png" src="../_images/image395.png" /></p>
<p>Overexposure to the sun is still dangerous, because there is a limit to
how much radiation our melanin can absorb. Most commercial sunscreens
claim to offer additional protection from both UV-A and UV-B radiation:
UV-A refers to wavelengths between 315-400 nm, UV-B to shorter, more
harmful wavelengths between 280-315 nm. PABA (<em>para</em>-aminobenzoic acid)
was used in sunscreens in the past, but its relatively high polarity
meant that it was not very soluble in oily lotions, and it tended to
rinse away when swimming. Many sunscreens today contain, among other
active ingredients, a more hydrophobic derivative of PABA called
<a class="reference external" href="https://en.wikipedia.org/wiki/Padimate_O">Padimate O</a>.</p>
<p>fig 19b</p>
<p>(<em>J. Investigative Derm</em>. <strong>1991</strong>, <em>97</em>, 340; <em>Photchem Photobiol</em>.
<strong>1982</strong>, <em>35</em>, 581)</p>
</section>
<section id="b-looking-at-uv-vis-spectra">
<h3>4.4B: Looking at UV-vis spectra<a class="headerlink" href="#b-looking-at-uv-vis-spectra" title="Permalink to this headline">#</a></h3>
<p>We have been talking in general terms about how molecules absorb UV and
visible light – now let’s look at some actual examples of data from a
UV-vis absorbance spectrophotometer. The setup is conceptually the same
as for IR spectroscopy: radiation with a range of wavelengths is
directed through a sample of interest, and a detector records which
wavelengths were absorbed and to what extent the absorption occurred.
Below is the absorbance spectrum of an important biological molecule
called nicotinamide adenine dinucleotide, abbreviated NAD<sup>+</sup>.
The adenine nucleotide base in the compound absorbs light in the
neighborhood of 260 nm, which is within the UV range of the
electromagnetic spectrum.</p>
<img alt="../_images/image396.png" src="../_images/image396.png" />
<p>fig 16</p>
<p>The convention in UV-vis spectroscopy is to show the baseline at the
bottom of the graph with the absorbance bands pointing up (opposite to
the way IR spectra are normally plotted). Wavelength values on the
x-axis are generally displayed in nanometer (nm) units. On the y-axis is
a measure of how much light was absorbed at each wavelength, expressed
as <strong>absorbance (A)</strong>. The absorbance value contains the same
information as the ‘percent transmittance’ number usually used in IR
spectroscopy, just expressed in different terms. To calculate absorbance
at a given wavelength, the computer in the spectrophotometer simply
takes the intensity of light at that wavelength <em>before</em> it passes
through the sample (I<sub>0</sub>), divides this value by the intensity
of the same wavelength <em>after</em> it passes through the sample (I), then
computes the log<sub>10</sub> of that number.</p>
<p>equation 4.4 A = log (I<sub>0</sub>/I)</p>
<p>Note that in this expression the units cancel, so A is a unitless value.</p>
<p>Typically, there are two things that we look for and record from a
UV-Vis spectrum. The first is <strong>λ<sub>max</sub></strong><sub>,</sub> the
wavelength at maximal absorbance, which visually is the very top of the
broad absorbance band. We also record the value of A at λ<sub>max</sub>,
which depends on the concentration of the sample. The NAD<sup>+</sup>
spectrum above shows a maximal absorbance of 1.0 at 260 nm. In the next
section, we will see how we can use this data to calculate the
concentration of NAD<sup>+</sup> in the sample.</p>
<p><u>Exercise 4.11</u>: Express A = 1.0 in terms of percent transmittance
(%T, the unit usually used in IR spectroscopy (and sometimes in UV-vis
as well).</p>
<p>Here is the absorbance spectrum of the common food coloring Red #3:</p>
<p><img alt="../_images/image397.png" src="../_images/image397.png" /></p>
<p>fig 17</p>
<p>Here, we see that the extended system of conjugated π bonds causes the
molecule to absorb light in the visible range - specifically, with a
λ<sub>max</sub> of 524 nm, which is in the blue/green range. Red
wavelengths pass through without being absorbed, so the solution appears
red to our eyes.</p>
<p>Now, take a look at the spectrum of another food coloring, Blue #1:</p>
<p><img alt="../_images/image398.png" src="../_images/image398.png" /></p>
<p>fig 18</p>
<p>Maximum absorbance is at 630 nm, in the orange range of the visible
spectrum, and the compound appears blue.</p>
</section>
<section id="d-applications-of-uv-spectroscopy-in-organic-and-biological-chemistry">
<h3>4.4D: Applications of UV spectroscopy in organic and biological chemistry<a class="headerlink" href="#d-applications-of-uv-spectroscopy-in-organic-and-biological-chemistry" title="Permalink to this headline">#</a></h3>
<p>UV-Visible spectroscopy has many applications in organic and biological
chemistry. The <strong>Beer - Lambert Law</strong> is often used to determine the
concentration of a UV-active sample. You most likely have performed a
Beer – Lambert experiment in a General Chemistry lab. The law is simply
an application of the observation that, within certain ranges, the
absorbance of UV-active compound at a given wavelength varies in a
linear fashion with its concentration: the higher the concentration, the
greater the absorbance.</p>
<img alt="../_images/image399.png" src="../_images/image399.png" />
<p>fig 31</p>
<p>If, for a particular sample, we divide the observed value of A at
λ<sub>max</sub> by the concentration of the sample (<em>c</em>, in mol/L), we
obtain the <strong>molar absorptivity</strong>, or <strong>extinction coefficient</strong> (ε),
which is a characteristic value for a given compound that we can look
up.</p>
<p>equation 4.5 ε = A/<em>cl</em></p>
<p>In the above equation, ‘<em>l</em> ‘ is <strong>path length</strong> - the distance that the
beam of light travels though the sample - expressed in cm. Most sample
holders are designed with a path length of1.0 cm, so the conventional
unit for molar absorptivity is L⋅mol<sup>-1</sup>⋅cm<sup>-1</sup>. In
all UV-Vis discussion or problems in this book, you should assume that
the relevant path length is 1.0 cm.</p>
<p>For example: In the spectrum of NAD<sup>+</sup> (section 4.3C), we see
that A = 1.0 at the λ<sub>max</sub> of 260 nm. We look up the molar
absorptivity of NAD<sup>+</sup> at 260 nm and find that it is listed as
ε(260) = 18,000 L⋅mol<sup>-1</sup>⋅cm<sup>-1</sup>. Using equation 4.4,
it is easy to calculate that the concentration of the sample is 5.6 x
10<sup>-5</sup> M.</p>
<p>The bases of DNA and RNA are good chromophores:</p>
<p><img alt="../_images/image400.png" src="../_images/image400.png" /></p>
<p>fig 20</p>
<p>Biochemists and molecular biologists often determine the concentration
of a DNA or oligonucleotide sample by assuming an average value of
ε(260) = 0.020 μL⋅ng<sup>-1</sup>⋅cm<sup>-1</sup> for double-stranded
DNA at its λ<sub>max</sub> of 260 nm (notice that concentration here is
expressed in terms of mass rather than moles: nanograms per microliter
is a convenient unit for DNA concentration when doing molecular
biology).</p>
<p><u>Exercise 4.12</u>: 50 μL of an aqueous sample of double stranded DNA
is dissolved in 950 μL of water. This diluted solution has a maximal
absorbance of 0.326 at 260 nm. What is the concentration of the original
(more concentrated) DNA sample, expressed in ug/μL?</p>
<p>Because the extinction coefficient of double stranded DNA is slightly
lower than that of single stranded DNA, we can use UV spectroscopy to
monitor a process known as DNA melting, an important factor in molecular
cloning experiments. If a short stretch of double stranded DNA is
gradually heated up, it will begin to ‘melt’, or break apart, as the
temperature increases (recall that two strands of DNA are held together
by a specific pattern of hydrogen bonds formed by ‘base-pairing’).</p>
<p><img alt="../_images/image401.png" src="../_images/image401.png" /></p>
<p>fig 21</p>
<p>As melting proceeds, the absorbance value for the sample increases,
eventually reaching a high plateau as all of the double-stranded DNA has
melted into two strands. The mid-point of this process, called the
‘melting temperature’, provides a good indication of how tightly the two
complementary strands of DNA are able to bind to each other.</p>
<p>In chapter 15 we will see how the Beer - Lambert Law and UV spectroscopy
provides us with a convenient way to follow the progress of many
different enzymatic redox (oxidation-reduction) reactions. In
biochemistry, oxidation of an organic molecule often occurs concurrently
with reduction of nicotinamide adenine dinucleotide (NAD<sup>+</sup>,
the compound whose spectrum we saw earlier in this section) to NADH:</p>
<p><img alt="../_images/image402.png" src="../_images/image402.png" /></p>
<p>fig 22</p>
<p>Proteins absorb light in the UV range due to the presence of the
aromatic amino acids tryptophan, phenylalanine, and tyrosine, all of
which are chromophores. Many coenzymes are also UV-active.</p>
<p><img alt="../_images/image403.png" src="../_images/image403.png" /></p>
<p>fig 24</p>
<p>Biochemists have used UV spectroscopy to study conformational changes in
proteins - how they change shape in response to different conditions.
When a protein undergoes a conformational shift (due to binding of a
ligand, for example), the resulting change in the environment around an
aromatic amino acid or coenzyme chromophore can cause the UV spectrum of
the protein to be altered.</p>
<p>Note: It will be very helpful for you to look at many more real examples
of mass spectrometry and infrared /ultraviolet-visible spectroscopy data
than are provided in this chapter - you can do this at the Spectral
Database for Organic Compounds.</p>
</section>
</section>
<section id="key-concepts-for-review">
<h2>Key concepts for review<a class="headerlink" href="#key-concepts-for-review" title="Permalink to this headline">#</a></h2>
<p><u>Mass spectrometry:</u></p>
<p>Understand the fundamentals of an MS experiment.</p>
<p>In a given mass spectrum, recognize the molecular ion peak along with
M+1 and M+2 peaks and be able to explain what these are and why they are
there.</p>
<p>Recognize characteristic mass spectral data for chloro- and
bromo-alkanes (M+2 peaks in 1:1 or 3:1 ratios, respectively).</p>
<p><u>Spectrometry basics:</u></p>
<p>Be able to put the following regions of the electromagnetic spectrum in
order from shortest to longest wavelength: gamma radiation, x-rays,
ultraviolet, blue visible light, red visible light, infrared, microwave,
radio wave.</p>
<p>Be able to mathematically convert among the following: wavelength
expressed in meters, wavelength expressed in wavenumbers (cm-1),
frequency, and energy in kJ/mol of photons. Non-quantitatively, you
should be able to quickly recognize relationships between the above: in
other words, you should have internalized the idea that radiation with
shorter wavelengths corresponds to higher wavenumbers, higher frequency,
and higher energy.</p>
<p>Be able to describe, in ‘generic’ terms, a molecular spectroscopy
experiment, and understand its elements: a spectrum of wavelengths
passing into a sample, a quantum transition from a ground to an excited
state in the sample molecules with absorbance of corresponding
wavelength(s) of radiation, and detection of wavelengths absorbed along
with the intensity of each absorbance.</p>
<p><u>IR spectroscopy:</u></p>
<p>Understand the basic idea of a vibrational transition</p>
<p>Be able to identify IR-active and inactive functional groups, and
predict which functional groups will lead to more intense absorbance
(ie., those with greater dipole moments)</p>
<p>Based on an IR spectrum, be confident in predicting the presence (or
absence) of the functional groups listed in table x.</p>
<p><u>UV-visible spectroscopy</u>:</p>
<p>Understand the basic idea of an electronic (π → π*; <em>n</em> → π*)
transition.</p>
<p>Be able to recognize a chromophore.</p>
<p>Be able to predict trends in absorbance maxima based on degree of
conjugation. Be able to use the Beer-Lambert Law to do simple
calculations (eg. figuring out concentration of a compound based on
observed A and literature value of molar absorptivity).</p>
</section>
<section id="problems">
<h2>Problems<a class="headerlink" href="#problems" title="Permalink to this headline">#</a></h2>
<p><strong>P4.1:</strong> Which represents a higher energy frequency of electromagnetic
radiation, 1690</p>
<p>cm<sup>-1</sup> or 3400 cm<sup>-1</sup>? Express each of these in terms
of wavelength (meters) and frequency (Hz).</p>
<p><strong>P4.2:</strong> Calculate the value, in kJ/mol, of the ∆E associated with a
typical alkyne carbon-carbon stretching frequency at 2100
cm<sup>-1</sup>.</p>
<p><strong>P4.3:</strong> Explain how you could use IR spectroscopy to distinguish
between compounds I, II, and III.</p>
<img alt="../_images/image404.png" src="../_images/image404.png" />
<p><strong>P4.4:</strong> Explain how you could use IR spectroscopy to distinguish
between compounds A, B, and C.</p>
<img alt="../_images/image405.png" src="../_images/image405.png" />
<p><strong>P4.5:</strong> Explain how you could use IR spectroscopy to distinguish
between compounds X, Y, and Z.</p>
<p><img alt="../_images/image406.png" src="../_images/image406.png" /></p>
<p><strong>P4.6:</strong> A 0.725 mL aqueous solution of NADH shows an absorbance at 340
nm of A = 0.257. Express (in nanomole (nm) units) how much NADH is
contained in the sample.</p>
<p><strong>P4.7:</strong> A 1 mL enzymatic reaction mixture contains NADH as one of the
reactants, and has a starting A<sub>340</sub> = 0.345. After the
reaction is allowed to run for five minutes, the researcher records a
value of A<sub>340</sub> = 0.125. How many nm of NADH are used up per
minute, on average, in this reaction?</p>
<p><strong>P4.8:</strong> The reaction below is of a type that we will study in Chapter
11. While the two starting materials are only slightly colored, the
product is an intense orange-red. Account for this observation.</p>
<p><img alt="../_images/image407.png" src="../_images/image407.png" /></p>
<p><strong>P4.9:</strong> Which would be more useful in distinguishing the two compounds
shown below: IR or UV spectroscopy? Explain.</p>
<p><img alt="../_images/image408.png" src="../_images/image408.png" /></p>
<p><strong>P4.10:</strong> Which analytical technique – IR, UV, or MS - could best be
used to distinguish between the two compounds below? Explain.</p>
<p><img alt="../_images/image409.png" src="../_images/image409.png" /></p>
<p><strong>P4.11:</strong> One would expect the mass spectrum of cyclohexanone to show a
molecular ion peak at m/z = 98. However, the m/z = 98 peak in the
cyclohexanone spectrum is unusually abundant, compared to the molecular
ion peaks in the mass spectra of other ketones such as 2-hexanone or
3-hexanone. Explain.</p>
</section>
</section>

    <script type="text/x-thebe-config">
    {
        requestKernel: true,
        binderOptions: {
            repo: "binder-examples/jupyter-stacks-datascience",
            ref: "master",
        },
        codeMirrorConfig: {
            theme: "abcdef",
            mode: "python"
        },
        kernelOptions: {
            kernelName: "python3",
            path: "./OCBE_I"
        },
        predefinedOutput: true
    }
    </script>
    <script>kernelName = 'python3'</script>

              </div>
              
            </main>
            <footer class="footer-article noprint">
                
    <!-- Previous / next buttons -->
<div class='prev-next-area'>
    <a class='left-prev' id="prev-link" href="Chapter%203%20-%20Conformation%20and%20stereochemistry.html" title="previous page">
        <i class="fas fa-angle-left"></i>
        <div class="prev-next-info">
            <p class="prev-next-subtitle">previous</p>
            <p class="prev-next-title">Chapter 3: Conformation and stereochemistry</p>
        </div>
    </a>
    <a class='right-next' id="next-link" href="Chapter%205%20-%20Structure%20determination%2C%20part%20II%20-%20Nuclear%20magnetic%20resonance%20spectroscopy.html" title="next page">
    <div class="prev-next-info">
        <p class="prev-next-subtitle">next</p>
        <p class="prev-next-title">Chapter 5: Structure determination, part II: Nuclear magnetic resonance spectroscopy</p>
    </div>
    <i class="fas fa-angle-right"></i>
    </a>
</div>
            </footer>
        </div>
    </div>
    <div class="footer-content row">
        <footer class="col footer"><p>
  
    By Tim Soderberg<br/>
  
      &copy; Copyright 2022.<br/>
</p>
        </footer>
    </div>
    
</div>


      </div>
    </div>
  
  <!-- Scripts loaded after <body> so the DOM is not blocked -->
  <script src="../_static/scripts/pydata-sphinx-theme.js?digest=1999514e3f237ded88cf"></script>


  </body>
</html>